<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <link href='https://fonts.googleapis.com/css?family=Architects+Daughter' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">

    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <title>Bioinformatics-cheatsheet by pwwang</title>
  </head>

  <body>
    <header>
      <div class="inner">
        <h1>Bioinformatics-cheatsheet</h1>
        <h2>A cheat sheet for Bioinformatians.</h2>
        <a href="https://github.com/pwwang/Bioinformatics-cheatsheet" class="button"><small>View project on</small> GitHub</a>
      </div>
    </header>

    <div id="content-wrapper">
      <div class="inner clearfix">
        <section id="main-content">
          <h1>
<a id="bioinformatics-cheatsheet" class="anchor" href="#bioinformatics-cheatsheet" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Bioinformatics-cheatsheet</h1>

<p>A cheat sheet for Bioinformatians. <a href="https://pwwang.github.io/Bioinformatics-cheatsheet/">@Github Pages</a></p>





<h2>
<a id="table-of-contents" class="anchor" href="#table-of-contents" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Table of Contents</h2>

<ul>
<li>
<a href="#general-elements">General Elements</a>

<ul>
<li>
<a href="#dnagenegenome">DNA/Gene/Genome</a>

<ul>
<li><a href="#related-terms">Related Terms</a></li>
<li><a href="#genomesequence-databases">Genome/Sequence Databases</a></li>
<li><a href="#general-gene-databases">General Gene Databases</a></li>
<li><a href="#specializeddisease-associated-gene-databases">Specialized/Disease-associated Gene Databases</a></li>
<li><a href="#gene-prediction">Gene Prediction</a></li>
<li><a href="#promotertss-prediction">Promoter/TSS Prediction</a></li>
</ul>
</li>
<li>
<a href="#rna">RNA</a>

<ul>
<li><a href="#related-terms-1">Related Terms</a></li>
</ul>
</li>
<li>
<a href="#protein">Protein</a>

<ul>
<li><a href="#related-terms-2">Related Terms</a></li>
<li><a href="#proteinprotein-domain-databases">Protein/Protein Domain Databases</a></li>
</ul>
</li>
<li>
<a href="#enhancer">Enhancer</a>

<ul>
<li><a href="#related-terms-3">Related Terms</a></li>
<li><a href="#enhancer-databases">Enhancer Databases</a></li>
<li><a href="#enhancer-prediction">Enhancer Prediction</a></li>
</ul>
</li>
</ul>
</li>
<li>
<a href="#interactionsregulationsassociations">Interactions/Regulations/Associations</a>

<ul>
<li>
<a href="#transcription-factor---target">Transcription Factor - Target</a>

<ul>
<li><a href="#related-terms-4">Related Terms</a></li>
<li><a href="#transcription-factor-databases">Transcription Factor Databases</a></li>
<li><a href="#tfbstf-binding-motiftf-target-databases">TFBS/TF Binding Motif/TF Target Databases</a></li>
<li><a href="#protein-dna-interaction-detection-methods">Protein-DNA Interaction Detection Methods</a></li>
</ul>
</li>
<li>
<a href="#protein-proteinchemical-interaction">Protein-Protein/Chemical Interaction</a>

<ul>
<li><a href="#related-terms-5">Related Terms</a></li>
<li><a href="#protein-protein-interaction-databases">Protein-Protein Interaction Databases</a></li>
<li><a href="#protein-chemical-interaction-databases">Protein-Chemical Interaction Databases</a></li>
<li><a href="#ppi-detection-methods">PPI Detection Methods</a></li>
</ul>
</li>
<li><a href="#correlation-databases">Correlation Databases</a></li>
</ul>
</li>
<li>
<a href="#epigenetics">Epigenetics</a>

<ul>
<li>
<a href="#dna-methylation">DNA Methylation</a>

<ul>
<li><a href="#dna-methylation-detection-methods">DNA Methylation Detection Methods</a></li>
</ul>
</li>
<li><a href="#histone-modification">Histone Modification</a></li>
</ul>
</li>
<li>
<a href="#biological-processes">Biological Processes</a>

<ul>
<li>
<a href="#pathways">Pathways</a>

<ul>
<li><a href="#related-terms-6">Related Terms</a></li>
<li><a href="#pathway-databases">Pathway Databases</a></li>
<li><a href="#pathway-predictions">Pathway Predictions</a></li>
<li><a href="#pathwaynetwork-analysisvisualizers">Pathway/Network analysis/visualizers</a></li>
</ul>
</li>
</ul>
</li>
<li>
<a href="#drugchemicals">Drug/Chemicals</a>

<ul>
<li><a href="#drugsmall-molecule-database">Drug/Small Molecule Database</a></li>
</ul>
</li>
<li>
<a href="#mutations-and-diseases">Mutations and Diseases</a>

<ul>
<li><a href="#diseasephenotype-databasesontologies">Disease/Phenotype Databases/Ontologies</a></li>
<li>
<a href="#germline-mutations-and-genetic-diseases">Germline Mutations and Genetic Diseases</a>

<ul>
<li><a href="#related-terms-7">Related Terms</a></li>
<li><a href="#genetic-variantdisease-databases">Genetic Variant/Disease Databases</a></li>
<li><a href="#tools">Tools</a></li>
<li><a href="#mutation-protein-structure-studies">Mutation-Protein Structure Studies</a></li>
</ul>
</li>
<li>
<a href="#somatic-mutations-and-cancers">Somatic Mutations and Cancers</a>

<ul>
<li><a href="#related-terms-8">Related Terms</a></li>
<li><a href="#cancer-data-repositories">Cancer Data Repositories</a></li>
</ul>
</li>
</ul>
</li>
<li>
<a href="#next-generate-sequencing">Next Generate Sequencing</a>

<ul>
<li><a href="#techniques">Techniques</a></li>
<li><a href="#ngs-data-repositories">NGS Data Repositories</a></li>
<li>
<a href="#ngs-data-analysis">NGS Data Analysis</a>

<ul>
<li><a href="#read-simulation">Read Simulation</a></li>
<li><a href="#read-trimming">Read Trimming</a></li>
<li><a href="#de-duplication">De-Duplication</a></li>
<li><a href="#alignment">Alignment</a></li>
<li><a href="#quality-control">Quality Control</a></li>
<li><a href="#peak-callingdifferential-peak-calling">Peak Calling/Differential Peak Calling</a></li>
<li><a href="#differential-expressed-gene-calling">Differential Expressed Gene Calling</a></li>
<li><a href="#variant-calling">Variant Calling</a></li>
<li><a href="#variant-filtering">Variant Filtering</a></li>
<li><a href="#variant-annotators">Variant Annotators</a></li>
<li><a href="#variant-simulation">Variant Simulation</a></li>
<li><a href="#haplotype-estimation-tools">Haplotype Estimation Tools</a></li>
<li><a href="#ngs-datavariantgenome-visulizersbrowsersdiagrams">NGS Data/Variant/Genome Visulizers/Browsers/Diagrams</a></li>
<li><a href="#ngs-data-analysis-pipelineframework">NGS Data Analysis Pipeline/framework</a></li>
</ul>
</li>
</ul>
</li>
<li>
<a href="#file-formats">File Formats</a>

<ul>
<li><a href="#formats">Formats</a></li>
<li><a href="#tools-1">Tools</a></li>
</ul>
</li>
<li>
<a href="#mathstatistics">Math/Statistics</a>

<ul>
<li><a href="#distributions">Distributions</a></li>
<li><a href="#tests">Tests</a></li>
<li><a href="#machine-learning">Machine Learning</a></li>
</ul>
</li>
</ul>



<h2>
<a id="general-elements" class="anchor" href="#general-elements" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>General Elements</h2>

<h3>
<a id="dnagenegenome" class="anchor" href="#dnagenegenome" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>DNA/Gene/Genome</h3>

<h4>
<a id="related-terms" class="anchor" href="#related-terms" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>DNA</strong>: Deoxyribonucleic acid is a molecule that carries the genetic instructions used in the growth, development, functioning and reproduction of all known living organisms and many viruses. <a href="https://en.wikipedia.org/wiki/DNA">@Wiki</a>, <a href="https://ghr.nlm.nih.gov/primer/basics/dna">@NIH</a>
</li>
<li>
<strong>Gene</strong>: A gene is a locus (or region) of DNA which is made up of nucleotides and is the molecular unit of heredity. <a href="https://en.wikipedia.org/wiki/Gene">@Wiki</a>, <a href="https://ghr.nlm.nih.gov/primer/basics/gene">@NIH</a>
</li>
<li>
<strong>Promoter</strong>: In genetics, a promoter is a region of DNA that initiates transcription of a particular gene. Promoters are located near the transcription start sites of genes, on the same strand and upstream on the DNA (towards the 5' region of the sense strand). <a href="https://en.wikipedia.org/wiki/Promoter_(genetics)">@Wiki</a>
</li>
<li>
<strong>TSS - Transcription Start Side</strong>: The transcription start site is the location where transcription starts at the 5'-end of a gene sequence. <a href="https://en.wikiversity.org/wiki/Gene_transcriptions/Start_sites">@Wiki</a>
</li>
<li>
<strong>Expression (Gene expression)</strong>: Gene expression is the process by which information from a gene is used in the synthesis of a functional gene product. These products are often proteins, but in non-protein coding genes such as transfer RNA (tRNA) or small nuclear RNA (snRNA) genes, the product is a functional RNA. <a href="https://en.wikipedia.org/wiki/Gene_expression">@Wiki</a>, <a href="http://www.nature.com/scitable/topicpage/gene-expression-14121669">@Scitable</a>
</li>
<li>
<strong>Exon</strong>: An exon is any part of a gene that will become a part of the final mature RNA produced by that gene after introns have been removed by RNA splicing. <a href="https://en.wikipedia.org/wiki/Exon">@Wiki</a>
</li>
<li>
<strong>Intron</strong>: An intron is any nucleotide sequence within a gene that is removed by RNA splicing during maturation of the final RNA product. <a href="https://en.wikipedia.org/wiki/Intron">@Wiki</a>
</li>
</ul>

<h4>
<a id="genomesequence-databases" class="anchor" href="#genomesequence-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Genome/Sequence Databases</h4>

<ul>
<li>
<strong><a href="http://www.sequenceontology.org/">SO - Sequence Ontology</a></strong>: SO is a collaborative ontology project for the definition of sequence features used in biological sequence annotation.</li>
<li>
<strong><a href="www.ensembl.org/">Ensembl genome browser</a></strong>: Ensembl is a joint project between EMBL - EBI and the Sanger Institute to develop a software system which produces and maintains automatic annotation on selected eukaryotic genomes.</li>
<li>
<strong><a href="www.candidagenome.org/">Candida Genome Database</a></strong>: Resource for genomic sequence data and gene and protein information for Candida albicans.</li>
<li>
<strong><a href="http://www.wormbase.org">WormBase</a></strong>: Worm Base.</li>
<li>
<strong><a href="http://flybase.org">FlyBase</a></strong>: FlyBase: a database of Drosophila Genes &amp; Genomes.</li>
<li>
<strong><a href="http://www.informatics.jax.org/">MGI - Mouse Genome Informatics</a></strong>: MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease. </li>
<li>
<strong><a href="rgd.mcw.edu/">RGD - Rat Genome Database</a></strong>: The Rat Genome Database (RGD) is the premier site for genetic, genomic, phenotype, and disease data generated from rat research.</li>
<li>
<strong><a href="www.yeastgenome.org/">Saccharomyces Genome Database</a></strong>: The Saccharomyces Genome Database (SGD) provides comprehensive integrated biological information for the budding yeast Saccharomyces cerevisiae.</li>
</ul>

<h4>
<a id="general-gene-databases" class="anchor" href="#general-gene-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>General Gene Databases</h4>

<ul>
<li>
<strong><a href="http://www.h-invitational.jp">H-InvDB</a></strong>: H-Invitational Database (H-InvDB) is an integrated database of human genes and transcripts.</li>
<li>
<strong><a href="http://www.genome.jp/kegg/genes.html">KEGG GENES</a></strong>: KEGG GENES is a collection of gene catalogs for all complete genomes generated from publicly available resources, mostly NCBI RefSeq and GenBank.</li>
<li>
<strong><a href="http://www.genenames.org/">HGNC</a></strong>: HGNC is responsible for approving unique symbols and names for human loci, including protein coding genes, ncRNA genes and pseudogenes, to allow unambiguous scientific communication.</li>
<li>
<strong><a href="www.genecards.org/">GeneCards</a></strong>: GeneCards is a searchable, integrated, database of human genes that provides concise genomic related information, on all known and predicted human genes.</li>
<li>
<strong><a href="www.ncbi.nlm.nih.gov/gene">NCBI Gene</a></strong>: A portal to gene-specific content based on NCBI's RefSeq project, information from model organism databases, and links to other resources.</li>
<li>
<strong><a href="https://www.wikigenes.org/">WikiGenes</a></strong>: WikiGenes is a non-profit initiative to provide a global collaborative knowledge base for the life sciences, where authorship matters.</li>
<li>
<strong><a href="http://www.gencodegenes.org/">GENCODE</a></strong>: Encyclopedia of genes and gene variants.</li>
<li>
<strong><a href="http://amp.pharm.mssm.edu/Harmonizome/">Harmonizome</a></strong>: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. <a href="http://database.oxfordjournals.org/content/2016/baw100.full">@Ref</a>
</li>
</ul>

<h4>
<a id="specializeddisease-associated-gene-databases" class="anchor" href="#specializeddisease-associated-gene-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Specialized/Disease-associated Gene Databases</h4>

<ul>
<li>
<strong><a href="http://www.bioguo.org/CADgene">CADgene</a></strong>: Coronary Artery Disease Gene Database.</li>
<li>
<strong><a href="http://genomics.senescence.info/genes">GenAge</a></strong>: GenAge: The Ageing Gene Database.</li>
</ul>

<h4>
<a id="gene-prediction" class="anchor" href="#gene-prediction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Gene Prediction</h4>

<ul>
<li>
<strong><a href="http://bgf.genomics.org.cn/">BGF</a></strong>: It is a hidden Markov model (HMM) and dynamic programming based ab initio gene prediction program.</li>
</ul>

<h4>
<a id="promotertss-prediction" class="anchor" href="#promotertss-prediction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Promoter/TSS Prediction</h4>

<ul>
<li>
<strong><a href="http://pepper.molgenrug.nl/index.php/prokaryote-promoters">PePPER</a></strong>: Prediction of prokaryote promoters.</li>
<li>
<strong><a href="http://www.cbs.dtu.dk/services/Promoter/">Promoter2.0</a></strong>: Promoter2.0 predicts transcription start sites of vertebrate PolII promoters in DNA sequences.</li>
</ul>

<h3>
<a id="rna" class="anchor" href="#rna" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>RNA</h3>

<h4>
<a id="related-terms-1" class="anchor" href="#related-terms-1" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>RNA</strong>: Ribonucleic acid (RNA) is a polymeric molecule implicated in various biological roles in coding, decoding, regulation, and expression of genes. <a href="https://en.wikipedia.org/wiki/RNA">@Wiki</a>
</li>
<li>
<strong>3'-UTR</strong>: is the section of messenger RNA (mRNA) that immediately follows the translation termination codon. <a href="https://en.wikipedia.org/wiki/Three_prime_untranslated_region">@Wiki</a>
</li>
<li>
<strong>5'-UTR</strong>: The 5' untranslated region (5′ UTR) (also known as a Leader Sequence or Leader RNA) is the region of an mRNA that is directly upstream from the initiation codon. <a href="https://en.wikipedia.org/wiki/Five_prime_untranslated_region">@Wiki</a>
</li>
</ul>

<h3>
<a id="protein" class="anchor" href="#protein" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein</h3>

<h4>
<a id="related-terms-2" class="anchor" href="#related-terms-2" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>Protein</strong>: Proteins are large biomolecules, or macromolecules, consisting of one or more long chains of amino acid residues. <a href="https://en.wikipedia.org/wiki/Protein">@Wiki</a>
</li>
<li>
<strong>Translation</strong>: In molecular biology and genetics, translation is the process in which cellular ribosomes create proteins. In translation, messenger RNA (mRNA)—produced by transcription from DNA—is decoded by a ribosome to produce a specific amino acid chain, or polypeptide. <a href="https://en.wikipedia.org/wiki/Translation_(biology)">@Wiki</a>
</li>
</ul>

<h4>
<a id="proteinprotein-domain-databases" class="anchor" href="#proteinprotein-domain-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein/Protein Domain Databases</h4>

<ul>
<li>
<strong><a href="http://ipfam.sanger.ac.uk">iPfam</a></strong>: Protein families database of alignments and HMMs.</li>
<li>
<strong><a href="http://pir.georgetown.edu/pirwww/dbinfo/iproclass.shtml">iProClass</a></strong>: The iProClass database provides value-added information reports for UniProtKB and unique NCBI Entrez protein sequences in UniParc, with links to over 160 biological databases, including databases for protein families, functions and pathways, interactions, structures and structural classifications, genes and genomes, ontologies, literature, and taxonomy.</li>
<li>
<strong><a href="http://mistdb.com">MiST</a></strong>: The Microbial Signal Transduction database contains the signal transduction proteins for bacterial and archaeal genomes.</li>
<li>
<strong><a href="http://modbase.compbio.ucsf.edu/modbase-cgi/index.cgi">ModBase</a></strong>: ModBase is a database of comparative protein structure models, calculated by modeling pipeline <a href="http://salilab.org/modpipe">ModPipe</a>.</li>
<li>
<strong><a href="http://www.rcsb.org/">RCSB PDB</a></strong>: The PDB archive contains information about experimentally-determined structures of proteins, nucleic acids, and complex assemblies.</li>
<li>
<strong><a href="http://pepbank.mgh.harvard.edu">PepBank</a></strong>: PepBank is a database of peptides based on sequence text mining and public peptide data sources.</li>
<li>
<strong><a href="http://cse.unl.edu/%7Eprofess">PROFESS</a></strong>: PROFESS is a biology database system that integrates databases describing PROtein Functions, Evolution, Structures and Sequences.</li>
<li>
<strong><a href="http://dunbrack2.fccc.edu/protcid">ProtCID</a></strong>: PROTein Common Interfaces Database.</li>
<li>
<strong><a href="http://suba.plantenergy.uwa.edu.au">SUBA3</a></strong>: The SUBcellular localization database for Arabidopsis proteins.</li>
<li>
<strong><a href="http://bioinformatics.charite.de/synsysnet">SynSysNet</a></strong>: Synaptic Proteins Database.</li>
<li>
<strong><a href="http://mdl.shsmu.edu.cn/ASD">ASD</a></strong>: Allosteric Database.</li>
</ul>

<h3>
<a id="enhancer" class="anchor" href="#enhancer" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Enhancer</h3>

<h4>
<a id="related-terms-3" class="anchor" href="#related-terms-3" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>Enhancer</strong>: In genetics, an enhancer is a short (50-1500 bp) region of DNA that can be bound by proteins (activators) to increase the likelihood that transcription of a particular gene will occur. These proteins are usually referred to as transcription factors. <a href="https://en.wikipedia.org/wiki/Enhancer_(genetics)">@Wiki</a>
</li>
<li>
<strong>Super Enhancer</strong>: In genetics, a super-enhancer is a region of the mammalian genome comprising multiple enhancers that is collectively bound by an array of transcription factor proteins to drive transcription of genes involved in cell identity. <a href="https://en.wikipedia.org/wiki/Super-enhancer">@Wiki</a>, <a href="http://www.nature.com/ng/journal/v47/n1/full/ng.3167.html">@Nature Genetics</a>
</li>
</ul>

<h4>
<a id="enhancer-databases" class="anchor" href="#enhancer-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Enhancer Databases</h4>

<ul>
<li>
<strong><a href="enhancer.lbl.gov/">VISTA Enhancer Browser</a></strong>: The VISTA Enhancer Browser is a central resource for experimentally validated human and mouse noncoding fragments with gene enhancer activity as assessed in transgenic mice.<br>
</li>
<li>
<strong><a href="http://www.cbrc.kaust.edu.sa/dendb/">DENdb</a></strong>: DENdb is a centralized on-line repository of predicted enhancers derived from multiple human cell-lines.</li>
<li>
<strong><a href="http://bioinfo.au.tsinghua.edu.cn/dbsuper/">dbSUPER</a></strong>: dbSUPER is the first integrated and interactive database of super-enhancers.</li>
<li>
<strong><a href="http://sea.edbc.org/">SEA</a></strong>: a super-enhancer archive.</li>
<li>
<strong><a href="http://www.dcode.org/EI/">EI</a></strong>: Database of EI candidate tissue-specific enhancers: Predicting Tissue-Specific Enhancers in the Human Genome. <a href="http://www.genome.org/cgi/reprint/17/2/201">@Ref</a>
</li>
<li>
<strong><a href="http://www.enhanceratlas.org/">EnhancerAtlas</a></strong>: a resource for enhancer annotation and analysis in 105 human cell/tissue types. <a href="http://bioinformatics.oxfordjournals.org/content/early/2016/08/20/bioinformatics.btw495.abstract">@Ref</a>
</li>
</ul>

<h4>
<a id="enhancer-prediction" class="anchor" href="#enhancer-prediction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Enhancer Prediction</h4>

<ul>
<li>
<strong><a href="http://cbrc.kaust.edu.sa/deep/">DEEP</a></strong>: a general computational framework for predicting enhancers</li>
</ul>

<h2>
<a id="interactionsregulationsassociations" class="anchor" href="#interactionsregulationsassociations" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Interactions/Regulations/Associations</h2>

<h3>
<a id="transcription-factor---target" class="anchor" href="#transcription-factor---target" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Transcription Factor - Target</h3>

<h4>
<a id="related-terms-4" class="anchor" href="#related-terms-4" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>TF - Transcription Factor</strong>: Transcription factors are proteins that control which genes are turned on or off in the genome. They do so by binding to DNA and other proteins. <a href="https://en.wikipedia.org/wiki/Transcription_factor">@Wiki</a>, <a href="https://www.broadinstitute.org/education/glossary/transcription-factor">@BroadInstitute</a>, <a href="http://www.nature.com/scitable/definition/general-transcription-factor-transcription-factor-167">@Scitable</a>
</li>
<li>
<strong>PWM - Position Weight Matrix/PSWM/PSSM</strong>: A position weight matrix (PWM), also known as a position-specific weight matrix (PSWM) or position-specific scoring matrix (PSSM), is a commonly used representation of motifs (patterns) in biological sequences. <a href="https://en.wikipedia.org/wiki/Position_weight_matrix">@Wiki</a>
</li>
<li>
<strong>TFBS - Transcription Factor Binding Site/DNA Binding Site</strong>: DNA binding sites are a type of binding site found in DNA where other molecules may bind. <a href="https://en.wikipedia.org/wiki/DNA_binding_site">@Wiki</a>
</li>
<li>
<strong>DNA Sequence Motif</strong>: Sequence motifs are short, recurring patterns in DNA that are presumed to have a biological function. <a href="http://www.nature.com/nbt/journal/v24/n4/full/nbt0406-423.html">@Nature Biotechnology</a>, <a href="https://en.wikipedia.org/wiki/Sequence_motif">@Wiki</a>
</li>
<li>
<strong>Transcription</strong>: Transcription is the first step of gene expression, in which a particular segment of DNA is copied into RNA (mRNA) by the enzyme RNA polymerase. <a href="https://en.wikipedia.org/wiki/Transcription_(genetics)">@Wiki</a>
</li>
</ul>

<h4>
<a id="transcription-factor-databases" class="anchor" href="#transcription-factor-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Transcription Factor Databases</h4>

<ul>
<li>
<strong><a href="http://bioinfo.life.hust.edu.cn/AnimalTFDB/">AnimalTFDB</a></strong>: AnimalTFDB is a comprehensive database including classification and annotation of genome-wide transcription factors (TFs), transcription co-factors and chromatin remodeling factors in 65 animal genomes. </li>
<li>
<strong><a href="www.transcriptionfactor.org/">DBD</a></strong>: DBD is a database of predicted transcription factors in completely sequenced genomes.</li>
<li>
<strong><a href="http://planttfdb.cbi.pku.edu.cn/">PlantTFDB</a></strong>: Plant transcription factor database, a portal for the functional and evolutionary study of plant transcription factors.</li>
<li>
<strong><a href="www.tfcat.ca/">TFCat</a></strong>: TFCat: The curated catalog of mouse and human transcription factors.</li>
<li>
<strong><a href="http://genome.gsc.riken.jp/TFdb/">TFdb</a></strong>: The Mouse transcription factor database (TFdb) is a database containing mouse transcription factor genes and their related genes.</li>
</ul>

<h4>
<a id="tfbstf-binding-motiftf-target-databases" class="anchor" href="#tfbstf-binding-motiftf-target-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>TFBS/TF Binding Motif/TF Target Databases</h4>

<ul>
<li>
<strong><a href="http://dc2.cistrome.org/">Cistrome DB</a></strong>: Cistrome DB is a comprehensive resource of hg38 and mm10 ChIP-seq data collection. Here is a brief introduction about the workflow of <a href="http://cistrome.org/chilin/">ChiLin</a>.</li>
<li>
<strong><a href="http://www.collectf.org/browse/home/">CollecTF</a></strong>: CollecTF is a database of transcription factor binding sites (TFBS) in the Bacteria domain.</li>
<li>
<strong><a href="http://insulatordb.uthsc.edu/">CTCFBSDB</a></strong>: A database for CTCF binding sites and genome organization.</li>
<li>
<strong><a href="http://www.factorbook.org/">FactorBook</a></strong>: This website organizes the analysis results of ENCODE TF ChIP-seq data, integrated with other ENCODE data such as ChIP-seq of histone marks and nucleosome occupancy.</li>
<li>
<strong><a href="http://floresta.eead.csic.es/footprintdb/index.php">footprintDB</a></strong>: footprintDB is a web server for assigning putative cis DNA motifs to input transcription factors (TFs) and conversely for predicting which TFs that might recognize input DNA motifs.</li>
<li>
<strong><a href="http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl">hmChIP</a></strong>: hmChIP is a database of genome-wide chromatin immu-noprecipitation (ChIP) data in human and mouse.</li>
<li>
<strong><a href="http://hocomoco.autosome.ru/">HOCOMOCO</a></strong>: HOmo sapiens COmprehensive MOdel COllection (HOCOMOCO) contains transcription factor (TF) binding models represented as classic Position Weight Matrices (PWMs, also known as Position-Specific Scoring Matrices, PSSMs) and precalculated score thresholds.</li>
<li>
<strong><a href="http://homer.salk.edu/homer/motif/motifDatabase.html">HOMER Motif Database</a></strong>: This database is maintained as part of HOMER and is mostly based on the analysis of public ChIP-Seq data sets.</li>
<li>
<strong><a href="http://bioinfo.wilmer.jhu.edu/PDI/">hPDI</a></strong>: The hPDI database holds experimental protein-DNA interaction data for humans identified by protein microarray assays.</li>
<li>
<strong><a href="http://www.lbbc.ibb.unesp.br/htri">HTRIdb</a></strong>: Human Transcriptional Regulation Interaction Database.</li>
<li>
<strong><a href="jaspar.binf.ku.dk/">JASPAR</a></strong>: The high-quality transcription factor binding profile database.</li>
<li>
<strong><a href="http://genome.ufl.edu/mapper/">MAPPER</a></strong>: MAPPER is a platform for the computational identification of transcription factor binding sites (TFBSs) in multiple genomes, that combines TRANSFAC® and JASPAR data with the search power of profile hidden Markov models (HMMs).</li>
<li>
<strong><a href="http://motifmap.ics.uci.edu/">MotifMap</a></strong>: The MotifMap system provides comprehensive maps of candidate regulatory elements encoded in the genomes of model species using databases of transcription factor binding motifs, refined genome alignments, and a comparative genomic statistical approach - Bayesian Branch Length Score.</li>
<li>
<strong><a href="http://opossum.cisreg.ca/oPOSSUM3/">oPOSSUM</a></strong>: oPOSSUM is a web-based system for the detection of over-represented conserved transcription factor binding sites and binding site combinations in sets of genes or sequences.</li>
<li>
<strong><a href="http://swissregulon.unibas.ch/fcgi/sr/swissregulon">SwissRegulon</a></strong>: Swissregulon Database contains genome-wide annotations of regulatory sites.</li>
<li>
<strong><a href="http://rohslab.cmb.usc.edu/TFBSshape/">TFBSshape</a></strong>: TFBSshape provides DNA shape features for transcription factor binding sites (TFBSs) that in addtion to sequence features, usually in the form of position weight matrices (PWMs), characterize DNA binding specificities of transcription factors (TFs) from 23 different species.</li>
<li>
<strong><a href="http://www.biobase-international.com/product/transcription-factor-binding-sites">TRANSFAC</a></strong>: TRANSFAC® is a unique knowledge-base containing published data on eukaryotic transcription factors and miRNAs, their experimentally-proven binding sites, and regulated genes.</li>
<li>
<strong><a href="http://the_brain.bwh.harvard.edu/uniprobe/index.php">UniPROBE</a></strong>: The UniPROBE (Universal PBM Resource for Oligonucleotide Binding Evaluation) database hosts data generated by universal protein binding microarray (PBM) technology on the in vitro DNA binding specificities of proteins. </li>
</ul>

<h4>
<a id="protein-dna-interaction-detection-methods" class="anchor" href="#protein-dna-interaction-detection-methods" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein-DNA Interaction Detection Methods</h4>

<ul>
<li>
<strong>PBM - Protein Binding Microarray</strong>: //TODO</li>
<li>
<strong>ChIP</strong>: Chromatin Immunoprecipitation (ChIP) is a type of immunoprecipitation experimental technique used to investigate the interaction between proteins and DNA in the cell. <a href="https://en.wikipedia.org/wiki/Chromatin_immunoprecipitation">@Wiki</a>
</li>
<li>
<strong>ChIP-seq</strong>: ChIP-sequencing, also known as ChIP-seq, is a method used to analyze protein interactions with DNA. ChIP-seq combines chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing to identify the binding sites of DNA-associated proteins. <a href="https://en.wikipedia.org/wiki/ChIP-sequencing">@Wiki</a>
</li>
<li>
<strong>ChIP-chip</strong>: ChIP-chip (also known as ChIP-on-chip) is a technology that combines chromatin immunoprecipitation ('ChIP') with DNA microarray ("chip"). Like regular ChIP, ChIP-on-chip is used to investigate interactions between proteins and DNA in vivo. <a href="https://en.wikipedia.org/wiki/ChIP-on-chip">@Wiki</a>
</li>
</ul>

<h3>
<a id="protein-proteinchemical-interaction" class="anchor" href="#protein-proteinchemical-interaction" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein-Protein/Chemical Interaction</h3>

<h4>
<a id="related-terms-5" class="anchor" href="#related-terms-5" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>PPI - Protein-Protein Interaction</strong>: Protein–protein interactions (PPIs) refer to lasting or ephemeral physical contacts of high specificity established between two or more protein molecules as a result of biochemical events steered by electrostatic forces including the hydrophobic effect. <a href="https://en.wikipedia.org/wiki/Protein%E2%80%93protein_interaction">@Wiki</a>
</li>
</ul>

<h4>
<a id="protein-protein-interaction-databases" class="anchor" href="#protein-protein-interaction-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein-Protein Interaction Databases</h4>

<ul>
<li>
<strong><a href="http://2p2idb.cnrs-mrs.fr">2P2Idb</a></strong>: 2P2I<sub>DB</sub> is a hand-curated database dedicated to the structure of protein-protein complexes with known small molecule inhibitors.</li>
<li>
<strong><a href="http://3d-interologs.life.nctu.edu.tw/index.php">3D-Interologs</a></strong>: The 3D-Interologs is a cross-species interacting database inferring from three-dimensional (3D) protein structure complexes and a novel scoring function by using 3D-domain interologs.</li>
<li>
<strong><a href="http://3did.irbbarcelona.org">3DID</a></strong>: The database of three-dimensional interacting domains (3did) is a collection of high-resolution three-dimensional structural templates for domain-domain interactions.</li>
<li>
<strong><a href="http://gmdd.shgmo.org/Computational-Biology/ANAP">ANAP</a></strong>: Arabidopsis Network Analysis Pipeline.</li>
<li>
<strong><a href="http://www.ddg-pharmfac.net/antijen/AntiJen/antijenhomepage.htm">AntiJen</a></strong>: AntiJen v2.0, is a database containing quantitative binding data for peptides binding to MHC Ligand, TCR-MHC Complexes, T Cell Epitope, TAP , B Cell Epitope molecules and immunological Protein-Protein interactions.</li>
<li>
<strong><a href="http://cicblade.dep.usal.es:8080/APID/init.action">APID</a></strong>: APID (Agile Protein Interactomes DataServer) provides a comprehensive collection of protein interactomes for more than 400 organisms based in the integration of known experimentally validated protein-protein physical interactions (PPIs).</li>
<li>
<strong><a href="http://wwwmgs.bionet.nsc.ru/mgs/gnw/aspd">ASPD</a></strong>: ASPD (Artificial Selected Proteins/Peptides Database) is a curated database on selected from randomized pools proteins and peptides.</li>
<li>
<strong><a href="http://protchem.hunnu.edu.cn/toxin/index.jsp">ATDB</a></strong>:ATDB mainly focuses on construct a globe-scale animal toxin-channel interaction network based on literatures and database annotations.</li>
<li>
<strong><a href="http://www.megabionet.org/atpid/webfile">AtPID</a></strong>: Arabidopsis thaliana Protein Interactome Database.</li>
<li>
<strong><a href="http://128.100.134.188/bacteriome">Bacteriome.org</a></strong>: Bacterial Protein Interaction Database for Escherichia Coli.</li>
<li>
<strong><a href="http://sbi.imim.es/web/index.php/research/servers/biana?page=biana.server">BIANA</a></strong>: Biologic Interaction and Network Analysis.</li>
<li>
<strong><a href="http://tsailab.chem.pacific.edu/wikiBID/index.php/Main_Page">BID</a></strong>: Binding Interface Database.</li>
<li>
<strong><a href="thebiogrid.org/">BioGRID</a></strong>: BioGRID Is An Online Interaction Respository With Data Compiled Through Comprehensive Curation Efforts.</li>
<li>
<strong><a href="http://bisc.cse.ucsc.edu">BISC</a></strong>: BISC(BInary SubComplex Database) is a new protein-protein interaction (PPI) database intending to bridge between the two communities most active in their characterisation: structural biology and functional genomics researchers.</li>
<li>
<strong><a href="http://interactome.dfci.harvard.edu">CCSB Interactome Database</a></strong>: Center for Cancer Systems Biology Interactome Database.</li>
<li>
<strong><a href="http://antares.protres.ru/comsin">ComSim</a></strong>: Database of protein structures in bound (Complex) and unbound (Single) states.</li>
<li>
<strong><a href="http://mips.helmholtz-muenchen.de/genre/proj/corum/index.html">CORUM</a></strong>: Comprehensive resource of mammalian protein complexes.</li>
<li>
<strong><a href="http://calcium.uhnres.utoronto.ca/ctdb/flash.htm">CTDB</a></strong>: Calmodulin Target Database.</li>
<li>
<strong><a href="http://cutdb.burnham.org">CutDB</a></strong>: CutDB: Proteolytic Event Database.</li>
<li>
<strong><a href="http://www.deathdomain.org">DeathDomain</a></strong>: A manually curated database of protein-protein interactions for Death Domain Superfamily.</li>
<li>
<strong><a href="http://webclu.bio.wzw.tum.de/dima/index.jsp">DIMA</a></strong>: DIMA is a Domain Interaction MAp and aims at becoming a comprehensive resource for functional and physical interactions among conserved protein-domains.</li>
<li>
<strong><a href="http://dip.doe-mbi.ucla.edu/">DIP</a></strong>: The DIP<sup>TM</sup> database catalogs experimentally determined interactions between proteins.</li>
<li>
<strong><a href="http://domine.utdallas.edu/cgi-bin/Domine">DOMINE</a></strong>: DOMINE is a database of known and predicted protein domain (domain-domain) interactions.</li>
<li>
<strong><a href="http://mint.bio.uniroma2.it/domino">DOMINO</a></strong>: DOMINO is an open-access database comprising more than 3900 annotated experiments describing interactions mediated by protein-interaction domains.</li>
<li>
<strong><a href="http://orion.rnet.missouri.edu/%7Enz953/DOMMINO">DOMMINO</a></strong>: Database of MacroMolecular Interactions .</li>
<li>
<strong><a href="http://www.droidb.org/">DroID</a></strong>: DroID is a comprehensive gene and protein interactions (interactome) database designed specifically for the model organism Drosophila.</li>
<li>
<strong><a href="http://dropnet.isima.fr/DroPNet_project/index.faces">DroPNet</a></strong>: Drosophila Protein Network.</li>
<li>
<strong><a href="http://ecid.bioinfo.cnio.es">EciD</a></strong>: E. coli Interaction Database.</li>
<li>
<strong><a href="http://funcoup.sbc.su.se">FunCoup</a></strong>: FunCoup is a framework to infer genome-wide functional couplings in 11 model organisms.</li>
<li>
<strong><a href="http://gene3d.biochem.ucl.ac.uk">Gene3D</a></strong>: Gene3D takes CATH domain families (from PDB structures) and assigns them to the millions protein sequences (using Hidden Markov models generated from HMMER) with no PDB structures.</li>
<li>
<strong><a href="http://biophysics.biol.uoa.gr/gpDB/index.jsp">gpDB</a></strong>: a database of GPCRs, G-proteins, Effectors and their interactions.</li>
<li>
<strong><a href="http://gwidd.bioinformatics.ku.edu">GWIDD</a></strong>: Genome-WIde protein Docking Database.</li>
<li>
<strong><a href="http://nesg.org:9090/HCPIN">HCPIN</a></strong>: Human Cancer Protein Interaction Network.</li>
<li>
<strong><a href="http://cbrc.kaust.edu.sa/hcvpro">HCVpro</a></strong>: Hepatitus C Virus Protein Interaction Database.</li>
<li>
<strong><a href="http://hint.yulab.org">HINT</a></strong>:HINT (High-quality INTeractomes) is a database of high-quality protein-protein interactions in different organisms.</li>
<li>
<strong><a href="http://hintdb.hgc.jp/htp">HitPredict</a></strong>: HitPredict is a resource of experimentally determined protein-protein interactions with reliability scores.</li>
<li>
<strong><a href="http://www.ncbi.nlm.nih.gov/RefSeq/HIVInteractions">HIV-1 Human Interaction Database</a></strong>: The HIV-1, human interactions project collates published reports of two types of interactions - protein interactions, and human gene knock-downs that affect virus replication and infectivity (reported as 'replication interactions').</li>
<li>
<strong><a href="http://www.hiv.lanl.gov/content/immunology">HIVMID</a></strong>: HIV Molecular Immunology Database.</li>
<li>
<strong><a href="http://prism.ccbb.ku.edu.tr/hotregion">HotRegion</a></strong>: A Database of Cooperative Hotspots.</li>
<li>
<strong><a href="http://dpi.nhri.org.tw">HP-DPI</a></strong>: Helicobacter pylori Database of Protein Interactomes.</li>
<li>
<strong><a href="http://www.hpid.org">HPID</a></strong>: Human Protein Interaction Database.</li>
<li>
<strong><a href="http://agbase.msstate.edu/hpi/main.html">HPIDB</a></strong>: Host-Pathogen Interaction Database.</li>
<li>
<strong><a href="http://www.hprd.org/">HPRD</a></strong>: The Human Protein Reference Database represents a centralized platform to visually depict and integrate information pertaining to domain architecture, post-translational modifications, interaction networks and disease association for each protein in the human proteome.</li>
<li>
<strong><a href="http://bioinformatics.biol.uoa.gr/human_gpdb">Human-gpDB</a></strong>: A database of human GPCRs, G-proteins, Effectors and their interactions.</li>
<li>
<strong><a href="http://www.proteome.com/control/researchproducts/insilico/proteome/details">HumanPSD</a></strong>: Human Proteome Survey Database .</li>
<li>
<strong><a href="http://hupi.ircm.qc.ca/hupi">HuPI</a></strong>: database of the Human Proteotheque Initiative.</li>
<li>
<strong><a href="http://ophid.utoronto.ca">I2D</a></strong>: Interologous Interaction Database.</li>
<li>
<strong><a href="http://www.ncbi.nlm.nih.gov/Structure/ibis/ibis.cgi">IBIS</a></strong>: Inferred Biomolecular Interactions (protein-protein, protein-small molecule, protein nucleic acids and protein-ion interactions) Server.</li>
<li>
<strong><a href="http://icbs.ics.uci.edu">ICBS</a></strong>: A database of protein-protein interactions mediated by interchain ß-sheet formation.</li>
<li>
<strong><a href="http://www.imexconsortium.org">IMEx</a></strong>: The International Molecular Exchange Consortium.</li>
<li>
<strong><a href="http://caps.ncbs.res.in/imotdb">iMOTdb</a></strong>: Interacting motifs in proteins database.</li>
<li>
<strong><a href="http://www.innatedb.ca">InnateDB</a></strong>: A Knowledge Resource for Innate Immunity Interactions and Pathways.</li>
<li>
<strong><a href="http://instruct.yulab.org">INstruct</a></strong>: a database of 3D protein interactome networks with structural resolution.</li>
<li>
<strong><a href="www.ebi.ac.uk/intact/">IntAct</a></strong>: IntAct provides a freely available, open source database system and analysis tools for molecular interaction data.</li>
<li>
<strong><a href="http://interactome-cmp.ucsf.edu/">Interactome</a></strong>: Krogan Lab Interactome Database.</li>
<li>
<strong><a href="http://interdom.i2r.a-star.edu.sg">InterDom</a></strong>: InterDom is a database of putative interacting protein domains derived from multiple sources, ranging from domain fusions (Rosetta Stone), protein interactions (DIP and BIND), protein complexes (PDB), to scientific literature (MEDLINE). </li>
<li>
<strong><a href="http://biodev.cea.fr/interevol">InterEvol</a></strong>: InterEvol database is designed for the analysis of co-evolution events at the interface of known structures of hetero- and homo-oligomers.</li>
<li>
<strong><a href="http://home.ku.edu.tr/%7Eokeskin/INTERFACE/INTERFACES.html">Interfaces</a></strong>: DATASET OF PROTEIN-PROTEIN INTERFACES.</li>
<li>
<strong><a href="http://interolog.gersteinlab.org">Interolog</a></strong>: Interolog/Regulog Database.</li>
<li>
<strong><a href="http://biodev.extra.cea.fr/interoporc">InteroPorc</a></strong>: InteroPorc is an automatic prediction tool to infer protein-protein interaction networks.</li>
<li>
<strong><a href="http://irefindex.org/wiki/index.php?title=iRefIndex">iRefIndex</a></strong>: iRefIndex provides an index of protein interactions available in a number of primary interaction databases including BIND, BioGRID, CORUM, DIP, HPRD, InnateDB, IntAct, MatrixDB, MINT, MPact, MPIDB and MPPI.</li>
<li>
<strong><a href="http://wodaklab.org/iRefWeb">iRefWeb</a></strong>: Interaction Reference Index Web Interface.</li>
<li>
<strong><a href="http://ir.hgc.jp">IRView</a></strong>: a database and viewer of interacting regions (IRs) in protein sequences.</li>
<li>
<strong><a href="http://matrixdb.univ-lyon1.fr/">MatrixDB</a></strong>: MatrixDB stores experimental data established by full-length proteins, matricryptins, glycosaminoglycans, lipids and cations.</li>
<li>
<strong><a href="http://mint.bio.uniroma2.it/">MINT</a></strong>: MINT focuses on experimentally verified protein-protein interactions mined from the scientific literature by expert curators.</li>
<li>
<strong><a href="http://mips.helmholtz-muenchen.de/proj/ppi">MIPS-MPPI</a></strong>: MIPS Mammalian Protein-Protein Interaction Database.</li>
<li>
<strong><a href="http://www.jcvi.org/mpidb/interaction.php?dbsource=MPI-LIT">MPI-LIT</a></strong>: the microbial protein interaction database.</li>
<li>
<strong><a href="http://bioinformatics.cau.edu.cn/cgi-bin/zzd-cgi/ppi/mpid.pl">MPID</a></strong>: Magnaporthe grisea Protein-protein Interaction Database.</li>
<li>
<strong><a href="http://variome.bic.nus.edu.sg/mpidt/index.html">MPID-T</a></strong>: MHC-Peptide Interaction Database.</li>
<li>
<strong><a href="http://www.jcvi.org/mpidb/about.php">MPIDB</a></strong>: Microbial Protein Interaction Database.</li>
<li>
<strong><a href="http://bio.ieo.eu/ncg2">NCG</a></strong>: NCG collects information on duplicability, orthology, evolutionary appearance and protein interactions network (PIN) properties of 736 cancer genes.</li>
<li>
<strong><a href="http://protein.cau.edu.cn/ncpi">NCPI</a></strong>: Neurospora Crassa Protein Interactome Database.</li>
<li>
<strong><a href="http://mips.helmholtz-muenchen.de/proj/ppi/negatome">Negatome</a></strong>: The Negatome is a collection of protein and domain pairs which are unlikely engaged in direct physical interactions.</li>
<li>
<strong><a href="http://cosbi.ku.edu.tr/prism/">PRISM</a></strong>: Protein Interactions by Structural Matching.</li>
<li>
<strong><a href="http://www.bioinsilico.org/cgi-bin/PCRPIDB/htmlPCRPI/home">PCRPi-DB</a></strong>: PCRPi-DB is a database of computationally annotated hot spots in protein interfaces.</li>
<li>
<strong><a href="http://icb.med.cornell.edu/services/pdz/start">PDZBase</a></strong>: PDZBase is a manually curated protein-protein interaction database developed specifically for interactions involving PDZ domains.</li>
<li>
<strong><a href="http://www-cryst.bioc.cam.ac.uk/%7Erichard/piccolo/about.php">PICCOLO</a></strong>: PICCOLO is a comprehensive database of structurally-characterized protein-protein interactions described at atomic level.</li>
<li>
<strong><a href="http://www.compbio.dundee.ac.uk/www-pips/">PIPs</a></strong>: PIPs is a database of predicted human protein-protein interactions. </li>
<li>
<strong><a href="http://pisite.hgc.jp">PiSITE</a></strong>: PiSITE is a web-based database of protein interaction sites.</li>
<li>
<strong><a href="http://protein.cau.edu.cn/ppira">PPIRA</a></strong>: Protein-Protein Interactions between Ralstonia solanacearum and Arabidopsis thaliana.</li>
<li>
<strong><a href="http://www.ebi.ac.uk/pdbe/pisa/">PDBePISA</a></strong>: PDBePISA is an interactive tool for the exploration of macromolecular interfaces.</li>
<li>
<strong><a href="http://bis.zju.edu.cn/prin">PRIN</a></strong>: Predicted Rice Interactome Database.</li>
<li>
<strong><a href="http://rkd.ucdavis.edu/interactome.shtml">RKD</a></strong>: Rice Kinase Database.</li>
<li>
<strong><a href="http://scoppi.biotec.tu-dresden.de/scoppi/index.html">SCOPPI</a></strong>: Structural classification of protein-protein interfaces.</li>
<li>
<strong><a href="http://www.scowlp.org/scowlp">SCOWLP</a></strong>: structural classification of protein binding reasons for atomic comparative analysis of protein interactions.</li>
<li>
<strong><a href="http://www.compbio.dundee.ac.uk/SNAPPI">SNAPPI-DB</a></strong>: Structures, iNterfaces and Alignments for Protein-Protein Interactions.</li>
<li>
<strong><a href="string-db.org/">STRING</a></strong>: functional protein association networks.</li>
<li>
<strong><a href="http://groups.csail.mit.edu/cb/struct2net/webserver">Struct2Net</a></strong>: Structure-based Computational Predictions of Protein-Protein Interactions.</li>
<li>
<strong><a href="http://www.syfpeithi.de">SYFPEITHI</a></strong>: Database of MHC Ligands and Peptide Motifs.</li>
<li>
<strong><a href="http://netbio.bgu.ac.il/tissuenet">TissueNet</a></strong>: The Database of Human Tissue Protein-Protein Interactions.</li>
<li>
<strong><a href="http://www.trpchannel.org">TRIP</a></strong>: a manually curated database of protein-protein interactions for mammalian TRP channels.</li>
<li>
<strong><a href="http://severus.dbmi.pitt.edu/wiki-pi">Wiki-Pi</a></strong>: Wiki-Pi: a wiki resource centred on human protein-protein interactions.</li>
<li>
<strong><a href="http://bioportal.kobic.re.kr/XooNET">XooNET</a></strong>: Integrated Protein-Protein Interaction database of Xanthomonas oryzae pathovar oryzae KACC1031.</li>
</ul>

<h4>
<a id="protein-chemical-interaction-databases" class="anchor" href="#protein-chemical-interaction-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Protein-Chemical Interaction Databases</h4>

<ul>
<li>
<strong><a href="http://potentia.cbs.dtu.dk/ChemProt/">ChemProt</a></strong>: The ChemProt 3.0 server is a ressource of annotated and predicted chemical-protein interactions. </li>
</ul>

<h4>
<a id="ppi-detection-methods" class="anchor" href="#ppi-detection-methods" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>PPI Detection Methods</h4>

<ul>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Co-immunoprecipitation">CoIP - Co-immunoprecipitation</a></strong>: is considered to be the gold standard assay for protein–protein interactions, especially when it is performed with endogenous (not overexpressed and not <a href="https://en.wikipedia.org/wiki/Epitope#Epitope_tags">tagged</a>) proteins. The protein of interest is isolated with a specific antibody. Interaction partners which stick to this protein are subsequently identified by <a href="https://en.wikipedia.org/wiki/Western_blotting">Western blotting</a>. Interactions detected by this approach are considered to be real. </li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Bimolecular_fluorescence_complementation">Bimolecular fluorescence complementation</a></strong>: (BiFC) is a new technique in observing the interactions of proteins. Combining with other new techniques, this method can be used to screen protein–protein interactions and their modulators, <a href="https://en.wikipedia.org/wiki/DERB">DERB</a>.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Affinity_electrophoresis">Affinity electrophoresis</a></strong>: as used for estimation of <a href="https://en.wikipedia.org/wiki/binding_constant">binding constant</a>s, as for instance in <a href="https://en.wikipedia.org/wiki/lectin">lectin</a> affinity electrophoresis or characterization of molecules with specific features like <a href="https://en.wikipedia.org/wiki/glycan">glycan</a> content or <a href="https://en.wikipedia.org/wiki/ligand">ligand</a> binding.</li>
<li>
<strong>Pull-down assays</strong>: are a common variation of <a href="https://en.wikipedia.org/wiki/immunoprecipitation">immunoprecipitation</a> and <a href="https://en.wikipedia.org/wiki/immunoelectrophoresis">immunoelectrophoresis</a> and are used identically, although this approach is more amenable to an initial screen for interacting proteins.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Label_transfer">Label transfer</a></strong>: can be used for screening or confirmation of protein interactions and can provide information about the interface where the interaction takes place. Label transfer can also detect weak or transient interactions that are difficult to capture using other ''in vitro'' detection strategies. In a label transfer reaction, a known protein is tagged with a detectable label. The label is then passed to an interacting protein, which can then be identified by the presence of the label.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/yeast_two-hybrid">Y2H - Yeast Two-Hybrid</a></strong>: Y2H screen investigates the interaction between artificial fusion proteins inside the nucleus of yeast. This approach can identify binding partners of a protein in an unbiased manner.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Phage_display">Phage display</a></strong>: used for the high-throughput screening of protein interactions</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Tandem_affinity_purification">TAP - Tandem Affinity Purification</a></strong>: (TAP) method allows high throughput identification of protein interactions. In contrast to yeast two-hybrid approach the accuracy of the method can be compared to those of small-scale experiments and the interactions are detected within the correct cellular environment as by <a href="https://en.wikipedia.org/wiki/co-immunoprecipitation">co-immunoprecipitation</a>. However, the TAP tag method requires two successive steps of protein purification and consequently it can not readily detect transient protein–protein interactions. </li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/cross-link%7CChemical_cross-linking">Cross-link/Chemical cross-linking</a></strong>: is often used to "fix" protein interactions in place before trying to isolate/identify interacting proteins. Common crosslinkers for this application include the non-cleavable <a href="https://en.wikipedia.org/wiki/N-Hydroxysuccinimide">NHS</a>-ester cross-linker, <a href="https://en.wikipedia.org/wiki/bissulfosuccinimidyl_suberate">bissulfosuccinimidyl suberate</a> (BS3); a cleavable version of BS3, <a href="https://en.wikipedia.org/wiki/dithiobis(sulfosuccinimidyl_propionate)">dithiobis(sulfosuccinimidyl propionate)</a> (DTSSP); and the <a href="https://en.wikipedia.org/wiki/imidoester">imidoester</a> cross-linker <a href="https://en.wikipedia.org/wiki/dimethyl_dithiobispropionimidate">dimethyl dithiobispropionimidate</a> (DTBP) that is popular for fixing interactions in <a href="https://en.wikipedia.org/wiki/ChIP">ChIP</a> assays.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/SPINE_(molecular_biology)">SPINE</a></strong>: (Strepprotein interaction experiment) uses a combination of reversible crosslinking with formaldehyde and an incorporation of an affinity tag to detect interaction partners ''in vivo''.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Quantitative_immunoprecipitation_combined_with_knock-down">Quantitative immunoprecipitation combined with knock-down</a></strong>: (QUICK) relies on co-immunoprecipitation, quantitative <a href="https://en.wikipedia.org/wiki/mass_spectrometry">mass spectrometry</a> (<a href="https://en.wikipedia.org/wiki/SILAC">SILAC</a>) and <a href="https://en.wikipedia.org/wiki/RNA_interference">RNA interference</a> (RNAi). This method detects interactions among endogenous non-tagged proteins. Thus, it has the same high confidence as co-immunoprecipitation. However, this method also depends on the availability of suitable antibodies.</li>
<li>
<strong><a href="https://en.wikipedia.org/wiki/Proximity_ligation_assay">Proximity ligation assay</a></strong>: (PLA) in situ is an immunohistochemical method utilizing so called PLA probes for detection of proteins, protein interactions and modifications.</li>
</ul>

<h3>
<a id="correlation-databases" class="anchor" href="#correlation-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Correlation Databases</h3>

<ul>
<li>
<strong><a href="https://cornet.psb.ugent.be">CORNET</a></strong>: CORrelation NETworks.</li>
<li>
<strong><a href="http://www.genemania.org">GeneMANIA</a></strong>: GeneMANIA finds other genes that are related to a set of input genes, using a very large set of functional association data.</li>
</ul>

<h2>
<a id="epigenetics" class="anchor" href="#epigenetics" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Epigenetics</h2>

<p>"The study of mitotically and/or meiotically heritable changes in gene function that cannot be explained by changes in DNA sequence." <a href="https://cshmonographs.org/index.php/monographs/issue/view/087969490.32">@Ref</a></p>

<h3>
<a id="dna-methylation" class="anchor" href="#dna-methylation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>DNA Methylation</h3>

<h4>
<a id="dna-methylation-detection-methods" class="anchor" href="#dna-methylation-detection-methods" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>DNA Methylation Detection Methods</h4>

<ul>
<li>
<strong>MeDIP/mDIP - Methylated DNA immunoprecipitation</strong>: Methylated DNA immunoprecipitation (MeDIP or mDIP) is a large-scale (chromosome- or genome-wide) purification technique in molecular biology that is used to enrich for methylated DNA sequences. <a href="https://en.wikipedia.org/wiki/Methylated_DNA_immunoprecipitation">@Wiki</a>
</li>
<li>
<strong>MeDIP-seq</strong>: The MeDIP-seq approach, i.e. the coupling of MeDIP with next generation, short-read sequencing technologies such as 454, Illumina (company) (Solexa), was first described by <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2644410">Down et al. in 2008</a>. The high-throughput sequencing of the methylated DNA fragments produces a large number of short reads (36-50bp or 400 bp depending on the technology). </li>
</ul>

<h3>
<a id="histone-modification" class="anchor" href="#histone-modification" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Histone Modification</h3>

<h2>
<a id="biological-processes" class="anchor" href="#biological-processes" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Biological Processes</h2>

<h3>
<a id="pathways" class="anchor" href="#pathways" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pathways</h3>

<h4>
<a id="related-terms-6" class="anchor" href="#related-terms-6" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>Pathway (Biological pathway)</strong>: A biological pathway is a series of actions among molecules in a cell that leads to a certain product or a change in a cell. Such a pathway can trigger the assembly of new molecules, such as a fat or protein. Pathways can also turn genes on and off, or spur a cell to move. <a href="https://en.wikipedia.org/wiki/Biological_pathway">@Wiki</a>, <a href="https://www.genome.gov/27530687/biological-pathways-fact-sheet/">@NIH</a>
</li>
</ul>

<h4>
<a id="pathway-databases" class="anchor" href="#pathway-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pathway Databases</h4>

<ul>
<li>
<strong><a href="http://consensuspathdb.org/">ConsensusPathDB</a></strong>: ConsensusPathDB is a database that integrates different types of functional interactions between physical entities in the cell like genes, RNA, proteins, protein complexes and metabolites in order to assemble a more complete and a less biased picture of cellular biology.</li>
<li>
<strong><a href="www.genome.jp/kegg/pathway.html">KEGG PATHWAY</a></strong>: KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction and reaction networks for different aspects.</li>
<li>
<strong><a href="http://metacyc.org/">MetaCyc</a></strong>: MetaCyc is a curated database of experimentally elucidated metabolic pathways from all domains of life.</li>
<li>
<strong><a href="http://www.informatics.jax.org/pathways.shtml">MouseCyc</a></strong>: MouseCyc is a database of curated biochemical pathways data for the laboratory mouse that can be integrated with functional and phenotypic data from MGI.</li>
<li>
<strong><a href="http://www.pantherdb.org/pathway/">PANTHER Pathway</a></strong>: PANTHER Pathway consists of over 177, primarily signaling, pathways, each with subfamilies and protein sequences mapped to individual pathway components. </li>
<li>
<strong><a href="http://pathwaycommons.org/">Pathway Commons</a></strong>: Pathway Commons aims to store and disseminate knowledge about biological pathways. Information is sourced from public pathway databases and is readily searched, visualized, and downloaded.</li>
<li>
<strong><a href="http://reactome.org/">Reactome</a></strong>: Reactome is a free, open-source, curated and peer reviewed pathway database.</li>
<li>
<strong><a href="http://www.plantcyc.org/">PLANTCYC</a></strong>: PlantCyc is a metabolic pathway reference database containing more than 800 pathways and their catalytic enzymes and genes, as well as compounds from over 350 plant species. It includes: <a href="http://www.plantcyc.org/databases/aracyc/14.0">AraCyc</a>(Arabidopsis thaliana col), <a href="http://www.plantcyc.org/databases/barleycyc/4.0">BarleyCyc</a>(Hordeum vulgare), <a href="http://www.plantcyc.org/databases/brachypodiumcyc/4.0">BrachypodiumCyc</a>(Brachypodium distachyon), <a href="http://www.plantcyc.org/databases/cassavacyc/6.0">CassavaCyc</a>(Manihot esculenta esculenta), <a href="http://www.plantcyc.org/databases/chinesecabbagecyc/4.0">ChineseCabbageCyc</a>(Brassica rapa ssp. pekinensis), <a href="http://www.plantcyc.org/databases/chlamycyc/6.0">ChlamyCyc</a>(Chlamydomonas reinhardtii), <a href="http://www.plantcyc.org/databases/corncyc/7.0">CornCyc</a>(Zea mays mays), <a href="http://www.plantcyc.org/databases/grapecyc/6.0">GrapeCyc</a>(Vitis vinifera), <a href="http://www.plantcyc.org/databases/mosscyc/5.0">MossCyc</a>(Physcomitrella patens), <a href="http://www.plantcyc.org/databases/oryzacyc/4.0">OryzaCyc</a>(Oryza sativa japonica group), <a href="http://www.plantcyc.org/databases/papayacyc/5.0">PapayaCyc</a>(Carica papaya), <a href="http://www.plantcyc.org/databases/poplarcyc/9.0">PoplarCyc</a>(Populus trichocarpa, other Populus species and hybrids), <a href="http://www.plantcyc.org/databases/potatocyc/3.0">PotatoCyc</a>(Solanum tuberosum), <a href="http://www.plantcyc.org/databases/selaginellacyc/5.0">SelaginellaCyc</a>(Selaginella moellendorffii), <a href="http://www.plantcyc.org/databases/setariacyc/4.0">SetariaCyc</a>(Setaria italica), <a href="http://www.plantcyc.org/databases/sorghumbicolorcyc/4.0">SorghumBicolorCyc</a>(Sorghum bicolor), <a href="http://www.plantcyc.org/databases/soycyc/7.0">SoyCyc</a>(Glycine max), <a href="http://www.plantcyc.org/databases/spirodelacyc/2.0">SpirodelaCyc</a>(Spirodela polyrhiza), <a href="http://www.plantcyc.org/databases/switchgrasscyc/4.0">SwitchgrassCyc</a>(Panicum virgatum), <a href="http://www.plantcyc.org/databases/tomatocyc/2.0">TomatoCyc</a>(Solanum lycopersicum), <a href="http://www.plantcyc.org/databases/wheatacyc/2.0">WheatACyc</a>(Triticum urartu), <a href="http://www.plantcyc.org/databases/wheatdcyc/2.0">WheatDCyc</a>(Aegilops tauschii)</li>
<li>
<strong><a href="http://signalink.org/">SignaLink</a></strong>: An integrated resource to analyze signaling pathway cross-talks, transcription factors, miRNAs and regulatory enzymes.</li>
<li>
<strong><a href="http://smpdb.ca/">SMPDB</a></strong>: SMPDB (The Small Molecule Pathway Database) is an interactive, visual database containing more than 618 small molecule pathways found in humans. More than 70% of these pathways (&gt;433) are not found in any other pathway database.</li>
<li>
<strong><a href="http://pathway.yeastgenome.org/">Yeast Pathways Database</a></strong>: The Yeast Pathways Database is a collection of manually curated metabolic pathways and enzymes of Saccharomyces cerevisiae.</li>
</ul>

<h4>
<a id="pathway-predictions" class="anchor" href="#pathway-predictions" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pathway Predictions</h4>

<ul>
<li>
<strong><a href="http://fraenkel-nsf.csbi.mit.edu/PIUMet/">PIUMet</a></strong>: Inferring Disease-Modifying Pathways and Hidden Components via Integrative Analysis of Metabolite Features with Various Omic Data. <a href="http://www.nature.com/nmeth/journal/v13/n9/pdf/nmeth.3940.pdf">@Ref</a>
</li>
</ul>

<h4>
<a id="pathwaynetwork-analysisvisualizers" class="anchor" href="#pathwaynetwork-analysisvisualizers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pathway/Network analysis/visualizers</h4>

<ul>
<li>
<strong><a href="https://github.com/raphael-group/hotnet2">HotNet2</a></strong>: HotNet2 is an algorithm for finding significantly altered subnetworks in a large gene interaction network. While originally developed for use with cancer mutation data, the current release also supports any application in which meaningful scores can be assigned to genes in the network. <a href="http://www.nature.com/ng/journal/v47/n2/full/ng.3168.html">@Ref</a>
</li>
<li>
<strong><a href="https://github.com/opencb/cell-maps/wiki">CellMaps</a></strong>: CellMaps is an open source HTML5 web-based application that allows researchers to easily model, visualize, integrate data and analyse biological networks inside a web browser. </li>
</ul>

<h2>
<a id="drugchemicals" class="anchor" href="#drugchemicals" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Drug/Chemicals</h2>

<h3>
<a id="drugsmall-molecule-database" class="anchor" href="#drugsmall-molecule-database" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Drug/Small Molecule Database</h3>

<ul>
<li>
<strong><a href="http://ahd.cbi.pku.edu.cn">AHD2.0</a></strong>: The aim of the Arabidopsis hormone database is to provide a systematic and comprehensive view of morphological phenotypes regulated by plant hormones, as well as regulatory genes participating in numerous plant hormone responses.</li>
<li>
<strong><a href="http://www.drugbank.ca/">DrugBank</a></strong>: The DrugBank database is a unique bioinformatics and cheminformatics resource that combines detailed drug (i.e. chemical, pharmacological and pharmaceutical) data with comprehensive drug target (i.e. sequence, structure, and pathway) information.</li>
<li>
<strong><a href="http://bidd.nus.edu.sg/BIDD-Databases/TTD/TTD.asp">TTD - Therapeutic Target Database</a></strong>: A database to provide information about the known and explored therapeutic protein and nucleic acid targets, the targeted disease, pathway information and the corresponding drugs directed at each of these targets.</li>
</ul>

<h2>
<a id="mutations-and-diseases" class="anchor" href="#mutations-and-diseases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Mutations and Diseases</h2>

<h3>
<a id="diseasephenotype-databasesontologies" class="anchor" href="#diseasephenotype-databasesontologies" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Disease/Phenotype Databases/Ontologies</h3>

<ul>
<li>
<strong><a href="http://human-phenotype-ontology.github.io/">HPO - Human Phenotype Ontoloy</a></strong>: The Human Phenotype Ontology (HPO) aims to provide a standardized vocabulary of phenotypic abnormalities encountered in human disease.</li>
<li>
<strong><a href="http://disease-ontology.org/">DO - Disease Ontology</a></strong>: The Disease Ontology semantically integrates disease and medical vocabularies through extensive cross mapping of DO terms to MeSH, ICD, NCI’s thesaurus, SNOMED and OMIM.</li>
<li>
<strong><a href="https://www.nlm.nih.gov/mesh/MBrowser.html">MeSH - Medical Subject Headings</a></strong>: MeSH is the National Library of Medicine's controlled vocabulary thesaurus. It consists of sets of terms naming descriptors in a hierarchical structure that permits searching at various levels of specificity.</li>
</ul>

<h3>
<a id="germline-mutations-and-genetic-diseases" class="anchor" href="#germline-mutations-and-genetic-diseases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Germline Mutations and Genetic Diseases</h3>

<h4>
<a id="related-terms-7" class="anchor" href="#related-terms-7" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>GWAS</strong>: In genetics, a genome-wide association study (GWA study, or GWAS), also known as whole genome association study (WGA study, or WGAS), is an examination of a genome-wide set of genetic variants in different individuals to see if any variant is associated with a trait. <a href="https://en.wikipedia.org/wiki/Genome-wide_association_study">@Wiki</a>
</li>
<li>
<strong>CNV - Copy-Number Variation</strong>: Copy-number variations (CNVs) are a form of structural variation that manifest as deletions or duplications in the genome. <a href="https://en.wikipedia.org/wiki/Copy-number_variation">@Wiki</a>(<a href="https://www.genome.gov/25520880/deoxyribonucleic-acid-dna-fact-sheet/">https://www.genome.gov/25520880/deoxyribonucleic-acid-dna-fact-sheet/</a>)</li>
<li>
<strong>SNP</strong>: A single nucleotide polymorphism, often abbreviated to SNP (pronounced snip; plural snips), is a variation in a single nucleotide that occurs at a specific position in the genome, where each variation is present to some appreciable degree within a population (e.g. &gt;1%). <a href="https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism">@Wiki</a>, <a href="https://ghr.nlm.nih.gov/primer/genomicresearch/snp">@NIH</a>
</li>
<li>
<strong>Genotype</strong>: The genotype is the part (DNA sequence) of the genetic makeup of a cell, and therefore of an organism or individual, which determines a specific characteristic (phenotype) of that cell/organism/individual. <a href="https://en.wikipedia.org/wiki/Genotype">@Wiki</a>
</li>
<li>
<strong>PheWAS</strong>: Phenome-wide association studies (PheWAS) analyze many phenotypes compared to a single genetic variant (or other attribute). <a href="https://github.com/PheWAS/PheWAS">@Tool</a>
</li>
<li>
<strong>Haplotype</strong>: A haplotype (haploid genotype) is a group of genes in an organism that are inherited together from a single parent. A haplogroup is a group of similar haplotypes that share a common ancestor with a single-nucleotide polymorphism mutation. <a href="https://en.wikipedia.org/wiki/Haplotype">@Wiki</a>, <a href="www.nature.com/scitable/definition/haplotype-haplotypes-142">@Scitable</a>
</li>
<li>
<strong>IBD - Identity By Descent</strong>: A DNA segment is identical by state (IBS) in two or more individuals if they have identical nucleotide sequences in this segment. An IBS segment is identical by descent (IBD) in two or more individuals if they have inherited it from a common ancestor without recombination, that is, the segment has the same ancestral origin in these individuals. <a href="https://en.wikipedia.org/wiki/Identity_by_descent">@Wiki</a>
</li>
<li>
<strong>Phasing/Haplotype Estimation</strong>: In genetics, haplotype estimation (also known as "phasing") refers to the process of statistical estimation of haplotypes from genotype data.</li>
<li>
<strong>ICD</strong>: The International Statistical Classification of Diseases and Related Health Problems, usually called by the short-form name International Classification of Diseases (ICD), is the international "standard diagnostic tool for epidemiology, health management and clinical purposes". <a href="https://en.wikipedia.org/wiki/International_Statistical_Classification_of_Diseases_and_Related_Health_Problems">@Wiki</a>
</li>
<li>
<strong>MAF - Minor Allele Frequency</strong>: MAF refers to the frequency at which the second most common allele occurs in a given population. SNPs with a minor allele frequency of 5% or greater were targeted by the HapMap project.</li>
<li>
<strong>LOH - Loss Of Heterozygosity</strong>: Loss of heterozygosity (LOH) is a gross chromosomal event that results in loss of the entire gene and the surrounding chromosomal region. <a href="https://en.wikipedia.org/wiki/Loss_of_heterozygosity">@Wiki</a>
</li>
<li>
<strong>Missense Mutation</strong>: A missense mutation is a point mutation in which a single nucleotide change results in a codon that codes for a different amino acid. <a href="https://en.wikipedia.org/wiki/Missense_mutation">@Wiki</a>
</li>
<li>
<strong>Nonsense Mutation</strong>: A nonsense mutation is a point mutation in a sequence of DNA that results in a premature stop codon, or a nonsense codon in the transcribed mRNA, and in a truncated, incomplete, and usually nonfunctional protein product.</li>
<li>
<strong>Rare Variant</strong>: A rare functional variant is a genetic variant which alters gene function, and which occurs at low frequency in a population. <a href="https://en.wikipedia.org/wiki/Rare_functional_variant">@Wiki</a>
</li>
<li>
<strong>Allele Frequency</strong>: Allele frequency, or gene frequency, is the relative frequency of an allele (variant of a gene) at a particular locus in a population, expressed as a fraction or percentage. <a href="https://en.wikipedia.org/wiki/Allele_frequency">@Wiki</a>
</li>
<li>
<strong>SV - Structural Variation</strong>: Structural variation (also genomic structural variation) is the variation in structure of an organism's chromosome. It consists of many kinds of variation in the genome of one species, and usually includes microscopic and submicroscopic types, such as deletions, duplications, copy-number variants, insertions, inversions and translocations.</li>
<li>
<strong><a href="https://www.mun.ca/biology/scarr/Transitions_vs_Transversions.html">Transition vs Transversion</a></strong>: DNA substitution mutations are of two types. Transitions are interchanges of two-ring purines (A &lt;-&gt; G) or of one-ring pyrimidines (C &lt;-&gt; T): they therefore involve bases of similar shape. Transversions are interchanges of purine for pyrimidine bases, which therefore involve exchange of one-ring and two-ring structures (A &lt;-&gt; C, A &lt;-&gt; T, G &lt;-&gt; C, G &lt;-&gt; T).</li>
</ul>

<h4>
<a id="genetic-variantdisease-databases" class="anchor" href="#genetic-variantdisease-databases" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Genetic Variant/Disease Databases</h4>

<ul>
<li>
<strong><a href="http://www.ncbi.nlm.nih.gov/SNP/">dbSNP</a></strong>: The Single Nucleotide Polymorphism Database (dbSNP) is a free public archive for genetic variation within and across different species developed and hosted by the National Center for Biotechnology Information (NCBI) in collaboration with the National Human Genome Research Institute (NHGRI).</li>
<li>
<strong><a href="http://jjwanglab.org/gwasdb">GWASdb</a></strong>: GWASdb is an online bioinformatics database combines collections of GVs from GWAS and their comprehensive functional annotations, as well as disease classifications.</li>
<li>
<strong><a href="http://www.gwascentral.org/">GWAS Central</a></strong>: GWAS Central provides a centralized compilation of summary level findings from genetic association studies, both large and small. </li>
<li>
<strong><a href="http://www.omim.org/">OMIM - Online Mendelian Inheritance in Man®</a></strong>: An Online Catalog of Human Genes and Genetic Disorders </li>
<li>
<strong><a href="https://emerge.mc.vanderbilt.edu/">eMERGE</a></strong>: eMERGE is a national network that combines DNA biorepositories with electronic medical record (EMR) systems for large scale, high-throughput genetic research in support of implementing genomic medicine.</li>
<li>
<strong><a href="https://www.genome.gov/10001688/international-hapmap-project/">International HapMap Project</a></strong>: The International HapMap Project was an organization that aimed to develop a haplotype map (HapMap) of the human genome, to describe the common patterns of human genetic variation. HapMap is used to find genetic variants affecting health, disease and responses to drugs and environmental factors.</li>
<li>
<strong><a href="http://www.mutdb.org/">MutDB</a></strong>: A database for assessing the impact of genetic variants. <a href="http://nar.oxfordjournals.org/content/36/suppl_1/D815">@Ref</a>
</li>
<li>
<strong><a href="http://www.nlgenome.nl/">The Genome of the Netherlands</a></strong>: 250 trios (father, mother and child) of Dutch descent.</li>
</ul>

<h4>
<a id="tools" class="anchor" href="#tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tools</h4>

<ul>
<li>
<strong><a href="http://bioconductor.org/packages/release/bioc/html/SNPRelate.html">SNPRelate</a></strong>: SNPRelate (high-performance computing R packages for multi-core symmetric multiprocessing computer architectures) to accelerate two key computations in GWAS: principal component analysis (PCA) and relatedness analysis using identity-by-descent (IBD) measures.</li>
<li>
<strong><a href="https://github.com/DReichLab/EIG">EIGENSOFT</a></strong>: The EIGENSOFT package combines functionality from our population genetics methods <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020190">@Ref</a>, EIGENSTRAT stratification correction method <a href="http://www.nature.com/ng/journal/v38/n8/abs/ng1847.html">@Ref</a>, and FastPCA and PC-based selection statistic <a href="http://www.sciencedirect.com/science/article/pii/S0002929716000033">@Ref</a>.</li>
<li>
<strong><a href="http://www.ritchielab.psu.edu/software/plato-download">PLATO</a></strong>: The PLatform for the Analysis, Translation, and Organization of large-scale data (PLATO) is a standalone program written in C++ that is designed to be a flexible and extensible analysis tool for a wide variety of genetic data.</li>
</ul>

<h4>
<a id="mutation-protein-structure-studies" class="anchor" href="#mutation-protein-structure-studies" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Mutation-Protein Structure Studies</h4>

<ul>
<li>
<strong><a href="https://github.com/ding-lab/hotspot3d">HotSpot3D</a></strong>: This 3D proximity tool can be used to identify the mutation hotspots in the linear 1D sequence and correlates these hotspots with known or potential interacting domains based on both known intermolecular interactions and calculated proximity for potential intramolecular interactions. <a href="http://www.nature.com/ng/journal/v48/n8/full/ng.3586.html">@Ref</a>
</li>
<li>
<strong><a href="http://mupit.icm.jhu.edu/MuPIT_Interactive/">MuPIT_Interactive</a></strong>: webserver for mapping variant positions to annotated, interactive 3D structures. <a href="http://link.springer.com/article/10.1007%2Fs00439-013-1325-0">@Ref</a>
</li>
<li>
<strong><a href="http://interactome3d.irbbarcelona.org/">Interactome3D</a></strong>: Interactome3D is a web service for the structural annotation of protein-protein interaction networks. <a href="http://www.nature.com/nmeth/journal/v10/n1/full/nmeth.2300.html">@Ref</a>
</li>
<li>
<strong><a href="https://github.com/KarchinLab/CLUMP">CLUMP</a></strong>: CLUMP (CLustering by Mutation Postion) is an unsupervised clustering of amino acid residue positions where variants occur, without any prior knowledge of their functional importance. <a href="http://www.pnas.org/content/112/40/E5486">@Ref</a>
</li>
</ul>

<h3>
<a id="somatic-mutations-and-cancers" class="anchor" href="#somatic-mutations-and-cancers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Somatic Mutations and Cancers</h3>

<h4>
<a id="related-terms-8" class="anchor" href="#related-terms-8" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Related Terms</h4>

<ul>
<li>
<strong>Cancer Predisposition Genes</strong>: Genes in which germline mutations confer highly or moderately increased risks of cancer. <a href="http://www.nature.com/nature/journal/v505/n7483/full/nature12981.html">@Nature</a>
</li>
</ul>

<h4>
<a id="cancer-data-repositories" class="anchor" href="#cancer-data-repositories" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Cancer Data Repositories</h4>

<ul>
<li>
<strong><a href="http://www.cbioportal.org/">cBioPortal</a></strong>: The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets. <a href="http://www.ncbi.nlm.nih.gov/pubmed/23550210">@Ref</a>, <a href="http://cancerdiscovery.aacrjournals.org/content/2/5/401.abstract">@Ref</a>
</li>
<li>
<strong><a href="http://cancergenome.nih.gov/">TCGA</a></strong>: The Cancer Genome Atlas (TCGA) is a collaboration between the National Cancer Institute (NCI) and the National Human Genome Research Institute (NHGRI) that has generated comprehensive, multi-dimensional maps of the key genomic changes in 33 types of cancer. </li>
<li>
<strong><a href="http://cancer.sanger.ac.uk/">COSMIC</a></strong>: COSMIC is an online database of somatically acquired mutations found in human cancer.</li>
<li>
<strong><a href="https://pecan.stjude.org/proteinpaint/">ProteinPaint</a></strong>: Explorer for genomic alteration in pediatric cancer. <a href="http://www.nature.com/ng/journal/v48/n1/full/ng.3466.html">@Ref</a>
</li>
</ul>

<h2>
<a id="next-generate-sequencing" class="anchor" href="#next-generate-sequencing" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Next Generate Sequencing</h2>

<h3>
<a id="techniques" class="anchor" href="#techniques" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Techniques</h3>

<ul>
<li>
<strong>DNA-seq</strong>: DNA sequencing is the process of determining the precise order of nucleotides within a DNA molecule. <a href="https://en.wikipedia.org/wiki/DNA_sequencing">@Wiki</a>
</li>
<li>
<strong>RNA-seq</strong>: RNA-seq (RNA sequencing), also called whole transcriptome shotgun sequencing<a href="WTSS">1</a>, uses next-generation sequencing (NGS) to reveal the presence and quantity of RNA in a biological sample at a given moment in time. <a href="https://en.wikipedia.org/wiki/RNA-Seq">@Wiki</a>
</li>
<li>
<strong>CLIP-Seq</strong>: <a href="https://en.wikipedia.org/wiki/CLIP">@Wiki</a>
</li>
<li>
<strong>FAIRE-seq</strong>: <a href="https://en.wikipedia.org/wiki/FAIRE-Seq">@Wiki</a>
</li>
<li>
<strong>DNase-Seq</strong>: <a href="https://en.wikipedia.org/wiki/DNase-Seq">@Wiki</a>
</li>
<li>
<strong>CAGE</strong>: <a href="https://en.wikipedia.org/wiki/Cap_analysis_gene_expression">@Wiki</a>
</li>
<li>
<strong>ChIA-PET</strong>: <a href="https://en.wikipedia.org/wiki/Chia_Pet">@Wiki</a>
</li>
<li>
<strong>5C/Hi-C</strong>: <a href="https://en.wikipedia.org/wiki/Chromosome_conformation_capture">@Wiki</a> </li>
</ul>

<h3>
<a id="ngs-data-repositories" class="anchor" href="#ngs-data-repositories" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>NGS Data Repositories</h3>

<ul>
<li>
<strong><a href="http://www.1000genomes.org/">1000 Genomes Project</a></strong>: The 1000 Genomes Project ran between 2008 and 2015, creating the largest public catalogue of human variation and genotype data.</li>
<li>
<strong><a href="http://www.ebi.ac.uk/arrayexpress/">Array Express</a></strong>: an NIH-funded database at the European Molecular Biology Laboratory -European Bioinformatics Institute that collects and disseminates microarray-based gene-expression data.</li>
<li>
<strong><a href="http://www.ddbj.nig.ac.jp/">DDBJ</a></strong>: DNA Data Bank of Japan (DDBJ) is a data bank organized by the National Institute of Genetics in Japan that collects sequence data.</li>
<li>
<strong><a href="https://www.encodeproject.org/">ENCODE</a></strong>: The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels, and regulatory elements that control cells and circumstances in which a gene is active.</li>
<li>
<strong><a href="http://www.ncbi.nlm.nih.gov/geo/">GEO</a></strong>: GEO is a public functional genomics data repository supporting MIAME-compliant data submissions. Array- and sequence-based data are accepted.</li>
<li>
<strong><a href="http://www.germonline.org/index.html">GermOnline</a></strong>: The GermOnline 4.0 gateway is a cross-species microarray expression database focusing on germline development, meiosis and gametogenesis as well as the mitotic cell cycle.</li>
<li>
<strong><a href="http://www.roadmapepigenomics.org/">Roadmap Epigenomics Project</a></strong>: The NIH Roadmap Epigenomics Mapping Consortium was launched with the goal of producing a public resource of human epigenomic data to catalyze basic biology and disease-oriented research.</li>
<li>
<strong><a href="http://www.ebi.ac.uk/gxa">Expression Atlas</a></strong>: The Expression Atlas provides information on gene expression patterns under different biological conditions such as a gene knock out, a plant treated with a compound, or in a particular organism part or cell. <a href="http://nar.oxfordjournals.org/content/44/D1/D746.full">@Ref</a>
</li>
</ul>

<h3>
<a id="ngs-data-analysis" class="anchor" href="#ngs-data-analysis" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>NGS Data Analysis</h3>

<h4>
<a id="read-simulation" class="anchor" href="#read-simulation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Read Simulation</h4>

<ul>
<li>
<strong><a href="https://sourceforge.net/p/readsim/wiki/Home/">ReadSim</a></strong>: ReadSim is a fast and simple reads simulator to target long reads such as PacBio or Nanopore.</li>
<li>
<strong><a href="http://www.ebi.ac.uk/goldman-srv/simNGS/">simNGS</a></strong>: simNGS is software for simulating observations from Illumina sequencing machines using the statistical models behind the AYB base-calling software.</li>
</ul>

<h4>
<a id="read-trimming" class="anchor" href="#read-trimming" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Read Trimming</h4>

<ul>
<li>
<strong><a href="http://www.usadellab.org/cms/index.php?page=trimmomatic">Trimmomatic</a></strong>: A flexible read trimming tool for Illumina NGS data.</li>
<li>
<strong><a href="https://github.com/najoshi/sickle">Sickle</a></strong>: A windowed adaptive trimming tool for FASTQ files using quality.</li>
<li>
<strong><a href="https://github.com/andreas-wilm/famas">famas</a></strong>: Yet another program for FastQ massaging with features: Quality- and length-based trimming, Random sampling, Splitting into multiple files, Order checking for paired-end files, Native gzip support.</li>
</ul>

<h4>
<a id="de-duplication" class="anchor" href="#de-duplication" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>De-Duplication</h4>

<ul>
<li>
<strong><a href="https://broadinstitute.github.io/picard/command-line-overview.html#MarkDuplicates">PiCard MarkDuplicates</a></strong>: This tool locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are defined as originating from a single fragment of DNA. </li>
<li>
<strong><a href="http://lomereiter.github.io/sambamba/docs/sambamba-markdup.html">sambamba-markdup</a></strong>: Find duplicate reads in BAM file.</li>
</ul>

<h4>
<a id="alignment" class="anchor" href="#alignment" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Alignment</h4>

<ul>
<li>
<strong><a href="http://bio-bwa.sourceforge.net/">bwa</a></strong>: BWA is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome.</li>
<li>
<strong><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">bowtie2</a></strong>: Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.</li>
<li>
<strong><a href="https://github.com/mozack/abra">ABRA</a></strong>: ABRA is a realigner for next generation sequencing data. It uses localized assembly and global realignment to align reads more accurately, thus improving downstream analysis (detection of indels and complex variants in particular). <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4173014/">@Ref</a>
</li>
<li>
<strong><a href="http://cibiv.github.io/NextGenMap/">NextGenMap</a></strong>: NextGenMap (NGM) is a flexible and fast read mapping program that is more than twice as fast as BWA, while achieving a mapping sensitivity similar to Stampy or Bowtie2. <a href="http://bioinformatics.oxfordjournals.org/content/29/21/2790.long">@Ref</a>
</li>
</ul>

<h4>
<a id="quality-control" class="anchor" href="#quality-control" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Quality Control</h4>

<ul>
<li>
<strong><a href="https://sourceforge.net/projects/clinqc/">ClinQC</a></strong>: ClinQC is an integrated and user-friendly pipeline for quality control, filtering and trimming of Sanger and NGS sequencing data for hundred to thousands of samples/patients in a single run in clinical research.</li>
<li>
<strong><a href="http://www.nipgr.res.in/ngsqctoolkit.html">NGS QC Toolkit</a></strong>: NGS QC Toolkit: A toolkit for the quality control (QC) of next generation sequencing (NGS) data.</li>
</ul>

<h4>
<a id="peak-callingdifferential-peak-calling" class="anchor" href="#peak-callingdifferential-peak-calling" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Peak Calling/Differential Peak Calling</h4>

<p><strong>Peak calling</strong> is a computational method used to identify areas in a genome that have been enriched with aligned reads as a consequence of performing a ChIP-sequencing or MeDIP-seq experiment. These areas are those where a protein interacts with DNA.<br>
<strong>Differential peak calling</strong> is about identifying significant differences in two ChIP-seq signals.</p>

<ul>
<li>
<strong><a href="http://liulab.dfci.harvard.edu/MACS/">MACS</a></strong>: Model-based analysis of ChIP-seq (MACS) is a computational algorithm that identifies genome-wide locations of transcription/chromatin factor binding or histone modification from ChIP-seq data. </li>
<li>
<strong><a href="http://pages.cs.wisc.edu/%7Ekliang/DBChIP/">DBChIP</a></strong>: detects differentially bound sharp binding sites across multiple conditions, with or without matching control samples.</li>
<li>
<strong><a href="http://bcb.dfci.harvard.edu/%7Egcyuan/MAnorm/MAnorm.htm">MAnorm</a></strong>: a robust model for quantitative comparison of ChIP-Seq data sets.</li>
<li>
<strong><a href="http://www.regulatory-genomics.org/thor-2/basic-intrstruction/">THOR</a></strong>: Differential peak calling of ChIP-seq signals with replicates. <a href="http://nar.oxfordjournals.org/content/early/2016/08/01/nar.gkw680.abstract">@Ref</a>
</li>
<li>
<strong><a href="http://www.regulatory-genomics.org/odin-2/basic-introduction/">ODIN</a></strong>: ODIN is an HMM-based approach to detect and analyse differential peaks in pairs of ChIP-seq data. ODIN performs genomic signal processing, peak calling and p-value calculation in an integrated framework. <a href="http://bioinformatics.oxfordjournals.org/content/30/24/3467">@Ref</a>
</li>
<li>
<strong><a href="https://www.bioconductor.org/packages/release/bioc/html/MMDiff.html">MMDiff</a></strong>: This package detects statistically significant difference between read enrichment profiles in different ChIP-Seq samples. <a href="bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-14-826">@Ref</a>
</li>
</ul>

<h4>
<a id="differential-expressed-gene-calling" class="anchor" href="#differential-expressed-gene-calling" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Differential Expressed Gene Calling</h4>

<ul>
<li>
<strong><a href="https://bioconductor.org/packages/release/bioc/html/limma.html">Limma</a></strong>: Linear Models for Microarray and RNA-Seq Data.</li>
<li>
<strong><a href="https://bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></strong>: Empirical Analysis of Digital Gene Expression Data in R.</li>
<li>
<strong><a href="http://bioconductor.org/packages/release/bioc/html/DESeq.html">DESeq</a></strong>: Differential gene expression analysis based on the negative binomial distribution.</li>
<li>
<strong><a href="http://cole-trapnell-lab.github.io/cufflinks/">Cufflinks</a></strong>: Transcriptome assembly and differential expression analysis for RNA-Seq.</li>
</ul>

<h4>
<a id="variant-calling" class="anchor" href="#variant-calling" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Calling</h4>

<ul>
<li>
<strong><a href="http://jjwanglab.org/fasd">FaSD</a></strong>: a fast and accurate single-nucleotide polymorphism detection program that uses a binomial distribution-based algorithm and a mutation probability.</li>
<li>
<strong><a href="http://soap.genomics.org.cn/soapsnp.html">SOAPsnp</a></strong>: SOAPsnp uses a method based on Bayes’ theorem (the reverse probability model) to call consensus genotype by carefully considering the data quality, alignment, and recurring experimental errors.</li>
<li>
<strong><a href="http://compbio.bccrc.ca/software/snvmix/">SNVmix</a></strong>: SNVMix is designed to detect single nucleotide variants from next generation sequencing data. </li>
<li>
<strong><a href="https://github.com/abyzovlab/CNVnator">CNVnator</a></strong>: a tool for CNV discovery and genotyping from depth-of-coverage by mapped reads.</li>
<li>
<strong><a href="https://samtools.github.io/bcftools/bcftools.html">bcftools</a></strong>: utilities for variant calling and manipulating VCFs and BCFs.</li>
<li>
<strong><a href="https://software.broadinstitute.org/gatk/">GATK</a></strong>: Genome Analysis Toolkit offers a wide variety of tools with a primary focus on variant discovery and genotyping.</li>
<li>
<strong><a href="https://cgwb.nci.nih.gov/goldenPath/bamview/documentation/index.html">Bambino</a></strong>: a variant detector and alignment viewer for next-generation sequencing data in the SAM/BAM format.</li>
<li>
<strong><a href="http://www.stjuderesearch.org/site/lab/zhang">CONSERTING</a></strong>: integrating copy-number analysis with structural-variation detection. <a href="http://www.nature.com/nmeth/journal/v12/n6/full/nmeth.3394.html">@Ref</a>
</li>
<li>
<strong><a href="http://www.stjuderesearch.org/site/lab/zhang">CREST</a></strong>: CREST (Clipping Reveals Structure) is a new algorithm for detecting genomic structural variations at base-pair resolution using next-generation sequencing data. <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.1628.html">@Ref</a>
</li>
<li>
<strong><a href="http://boevalab.com/FREEC/">Control-FREEC</a></strong>: Control-FREEC is a tool for detection of copy-number changes and allelic imbalances (including LOH) using deep-sequencing data.</li>
<li>
<strong><a href="http://compbio.bccrc.ca/software/hmmcopy/">HMMcopy</a></strong>: Copy number prediction with correction for GC and mappability bias for HTS data</li>
<li>
<strong><a href="http://portals.broadinstitute.org/cgi-bin/cancer/publications/pub_paper.cgi?mode=view&amp;paper_id=182">SegSeq</a></strong>: an algorithm to identify chromosomal breakpoints using massively parallel sequence data.</li>
<li>
<strong><a href="http://tiger.dbs.nus.edu.sg/cnv-seq/">CNV-seq</a></strong>: a new method to detect copy number variation using high-throughput sequencing.</li>
<li>
<strong><a href="http://www.math.pku.edu.cn/teachers/xirb/downloads/software/BICseq2/BICseq2.html">BICseq2</a></strong>: BICseq2 is an algorithm developed for the normalization of  high-throughput sequencing (HTS) data and detection of copy number variations (CNV) in the genome. BICseq2 can be used for detecting CNVs with or without a control genome.</li>
<li>
<strong><a href="http://bioinformatics.mdanderson.org/main/MuSE">MuSE</a></strong>: a novel approach to mutation calling based on the F81 Markov substitution model for molecular evolution, which models the evolution of the reference allele to the allelic composition of the matched tumor and normal tissue at each genomic locus. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1029-6">@Ref</a>
</li>
<li>
<strong><a href="http://dkoboldt.github.io/varscan/">VarScan</a></strong>: a platform-independent software tool developed at the Genome Institute at Washington University to detect variants in NGS data.</li>
<li>
<strong><a href="https://github.com/genome/pindel">Pindel</a></strong>: Pindel can detect breakpoints of large deletions, medium sized insertions, inversions, tandem duplications and other structural variants at single-based resolution from next-gen sequence data.</li>
<li>
<strong>COPS</strong>: A Sensitive and Accurate Tool for Detecting Somatic Copy Number Alterations Using Short-Read Sequence Data from Paired Samples. COPS is available at ftp://115.119.160.213 with username “cops” and password “cops”. <a href="http://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0047812">@Ref</a>
</li>
<li>
<strong><a href="https://bitbucket.org/joseph07/multisnv/wiki/Home">multiSNV</a></strong>: multiSNV is a tool for calling somatic single-nucleotide variants (SNVs) using NGS data from a normal and multiple tumour samples of the same patient. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4482059/">@Ref</a>
</li>
<li>
<strong><a href="http://bioinform.github.io/somaticseq/">SomaticSeq</a></strong>: SomaticSeq is a flexible post-somatic-mutation-calling workflow for improved accuracy. <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4574535/">@Ref</a>
</li>
<li>
<strong><a href="http://seselab.org/cosmos/">Cosmos</a></strong>: COSMOS can detect somatic structural variations from whole genome short-read sequences. <a href="http://nar.oxfordjournals.org/content/44/8/e78">@Ref</a>
</li>
</ul>

<h4>
<a id="variant-filtering" class="anchor" href="#variant-filtering" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Filtering</h4>

<ul>
<li>
<strong><a href="http://snpeff.sourceforge.net/SnpSift.html">SnpSift</a></strong>: SnpSift is a toolbox that allows you to filter and manipulate annotated files.</li>
<li>
<strong><a href="https://varapp-demo.vital-it.ch/#/login">Varapp</a></strong>: Varapp is an open-source web application to filter variants from large sets of exome data stored in a relational database. </li>
</ul>

<h4>
<a id="variant-annotators" class="anchor" href="#variant-annotators" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Annotators</h4>

<ul>
<li>
<strong><a href="http://annovar.openbioinformatics.org/en/latest/">ANNOVAR</a></strong>: is an efficient software tool to utilize update-to-date information to functionally annotate genetic variants detected from diverse genomes (including human genome hg18, hg19, hg38, as well as mouse, worm, fly, yeast and many others). </li>
<li>
<strong><a href="http://snpeff.sourceforge.net/">SnpEff</a></strong>: Genetic variant annotation and effect prediction toolbox. <a href="http://www.tandfonline.com/doi/abs/10.4161/fly.19695">@Ref</a>
</li>
<li>
<strong><a href="https://gemini.readthedocs.io/en/latest/">GEMINI</a></strong>: GEMINI (GEnome MINIng) is a flexible framework for exploring genetic variation in the context of the wealth of genome annotations available for the human genome. <a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003153">@Ref</a>
</li>
<li>
<strong><a href="http://asia.ensembl.org/Homo_sapiens/Tools/VEP?db=core">Variant Effect Predictor</a></strong>: Analyse your own variants and predict the functional consequences of known and unknown variants via our Variant Effect Predictor (VEP) tool. <a href="http://bioinformatics.oxfordjournals.org/content/26/16/2069">@Ref</a>
</li>
<li>
<strong><a href="http://vat.gersteinlab.org/index.php">VAT - Variant Annotation Tool</a></strong>: A computational framework to functionally annotate variants in personal genomes using a cloud-computing environment. <a href="http://bioinformatics.oxfordjournals.org/content/28/17/2267">@Ref</a>
</li>
<li>
<strong><a href="http://snp.gs.washington.edu/SeattleSeqAnnotation147/">SeattleSeq Variation Annotation</a></strong>: The SeattleSeq Annotation server provides annotation of SNVs (single-nucleotide variations) and small indels, both known and novel. <a href="http://www.nature.com/nature/journal/v461/n7261/full/nature08250.html">@Ref</a>
</li>
<li>
<strong><a href="https://github.com/charite/jannovar">Jannovar</a></strong>: A Java Library for Exome Annotation.</li>
<li>
<strong><a href="https://github.com/opencb/cellbase">Cellbase</a></strong>: CellBase is a scalable and high-performance NoSQL database that integrates relevant biological information from well-known data sources such as Ensembl, Uniprot, IntAct or ClinVar among others. All this data can be queried through a comprehensive RESTful web services API or using the command line interface. Also, a built-in variant annotator has been developed and can be used to annotate files containing variants in Variant Call Format (VCF). <a href="http://nar.oxfordjournals.org/content/40/W1/W609.short">@Ref</a>
</li>
<li>
<strong><a href="https://github.com/brinkmanlab/GenomeD3Plot/">GenomeD3Plot</a></strong>: GenomeD3Plot (formerly <a href="https://github.com/lairdm/islandplot">Islandplot</a>) is an SVG based genome viewer written in javascript using D3.</li>
<li>
<strong><a href="http://varianttools.sourceforge.net/Main/HomePage">Variant Tools</a></strong>: variant tools is a software tool for the manipulation, annotation, selection, simulation, and analysis of variants in the context of next-gen sequencing analysis. <a href="http://bioinformatics.oxfordjournals.org/content/28/3/421.abstract?sid=f64403e7-5050-4102-963c-e690efe003f7">@Ref</a> </li>
</ul>

<h4>
<a id="variant-simulation" class="anchor" href="#variant-simulation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Simulation</h4>

<ul>
<li>
<strong><a href="https://sourceforge.net/projects/scnvsim/">SCNVSim</a></strong>: somatic copy number variation and structure variation simulator. <a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0502-7">@Ref</a>
</li>
</ul>

<h4>
<a id="haplotype-estimation-tools" class="anchor" href="#haplotype-estimation-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Haplotype Estimation Tools</h4>

<ul>
<li>
<strong><a href="http://stephenslab.uchicago.edu/phase/download.html">PHASE</a></strong>: A program for reconstructing haplotypes from population data. PHASE was limited by its speed and was not applicable to datasets from genome-wide association studies. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275651/">@Ref</a> </li>
<li>
<strong><a href="http://scheet.org/code/fastphase_doc_1.4.pdf">fastPHASE</a></strong>: fastPHASE is software that implements methods for estimating missing genotypes and reconstructing haplotypes from unphased SNP genotype data of unrelated individuals.  <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1424677/">@Ref</a> </li>
<li>
<strong><a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html">IMPUTE2</a></strong>: IMPUTE version 2 (also known as IMPUTE2) is a genotype imputation and haplotype phasing program based on ideas from <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000529">Howie et al. 2009</a>
</li>
</ul>

<h4>
<a id="ngs-datavariantgenome-visulizersbrowsersdiagrams" class="anchor" href="#ngs-datavariantgenome-visulizersbrowsersdiagrams" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>NGS Data/Variant/Genome Visulizers/Browsers/Diagrams</h4>

<ul>
<li>
<strong><a href="https://genome.ucsc.edu/cgi-bin/hgTracks">UCSC Genome Browser</a></strong>: The UCSC Genome Browser is an on-line genome browser hosted by the University of California, Santa Cruz (UCSC).</li>
<li>
<strong><a href="http://jbrowse.org/">JBrowse</a></strong>: a JavaScript genome browser by the open-source Generic Model Organism Database project. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2752129">@Ref</a>
</li>
<li>
<strong><a href="http://visualization.ritchielab.psu.edu/synthesis_views/plot">Synthesis-View</a></strong>: A SNP visualization tool. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3012023/">@Ref</a>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/igv/">IGV - Integrative Genomics Viewer</a></strong>: A high-performance visualization tool for interactive exploration of large, integrated genomic datasets. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603213">@Ref</a>
</li>
<li>
<strong><a href="http://www.hammerlab.org/2015/06/19/introducing-pileup-js-a-browser-based-genome-viewer/">pileup.js</a></strong>: a Browser-based Genome Viewer.</li>
<li>
<strong><a href="http://www.biodalliance.org/index.html">Biodalliance</a></strong>: Biodalliance is a fast, interactive, genome visualization tool that's easy to embed in web pages and applications.</li>
<li>
<strong><a href="https://github.com/3DGenomes/TADkit">TADkit</a></strong>: TADkit is a HTML5 and JavaScript-based 3D genome browser. It makes use of D3.js for rendering the 1D and 2D tracks and WebGl by Three.js for rendering the 3D track. </li>
</ul>

<h4>
<a id="ngs-data-analysis-pipelineframework" class="anchor" href="#ngs-data-analysis-pipelineframework" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>NGS Data Analysis Pipeline/framework</h4>

<ul>
<li>
<strong><a href="https://www.nextflow.io/">nextflow</a></strong>: Nextflow is a fluent DSL modelled around the UNIX pipe concept, that simplifies writing parallel and scalable pipelines in a portable manner.</li>
<li>
<strong><a href="http://rubioseq.bioinfo.cnio.es/">RUbioSeq+</a></strong>: RUbioSeq+ is a stand-alone and multiplatform application for the integrated analysis of NGS data. More specifically, our software implements pipelines for the analysis of single nucleotide and copy-number variation, bisulfite-seq and ChIP-seq experiments using well-established tools to perform these common tasks.</li>
<li>
<strong><a href="https://github.com/hall-lab/speedseq">SpeedSeq</a></strong>: A flexible framework for rapid genome analysis and interpretation. <a href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.3505.html">@Ref</a>
</li>
</ul>

<h2>
<a id="file-formats" class="anchor" href="#file-formats" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>File Formats</h2>

<h3>
<a id="formats" class="anchor" href="#formats" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Formats</h3>

<ul>
<li>
<strong>BAM</strong>: BAM is the compressed binary version of the Sequence Alignment/Map (SAM) format, a compact and index-able representation of nucleotide sequence alignments.</li>
<li>
<strong>BED</strong>: The BED format consists of one line per feature, each containing 3-12 columns of data, plus optional track definition lines. <a href="https://genome.ucsc.edu/FAQ/FAQformat.html#format1">@UCSC</a>, <a href="http://bedtools.readthedocs.io/en/latest/content/general-usage.html">@bedtools</a>, <a href="http://useast.ensembl.org/info/website/upload/bed.html">@Ensembl</a>
</li>
<li>
<strong>BigBed</strong>: The bigBed format stores annotation items that can either be simple, or a linked collection of exons, much as BED files do. BigBed files are created initially from BED type files, using the program bedToBigBed. The resulting bigBed files are in an indexed binary format. <a href="https://genome.ucsc.edu/goldenpath/help/bigBed.html">@UCSC</a>
</li>
<li>
<strong>BigWig</strong>: The BigWig format is designed for dense, continuous data that is intended to be displayed as a graph. Files can be created from WIG or BedGraph files using the appropriate utility program. <a href="http://genome.ucsc.edu/goldenPath/help/bigWig.html">@UCSC</a>
</li>
<li>
<strong>GFF</strong>: The GFF (General Feature Format) format consists of one line per feature, each containing 9 columns of data, plus optional track definition lines. <a href="https://genome.ucsc.edu/FAQ/FAQformat.html#format3">@UCSC</a>, <a href="http://useast.ensembl.org/info/website/upload/gff.html">@Ensembl</a>, <a href="http://gmod.org/wiki/GFF2">GTF(GFFv2)@GMOD</a>, <a href="https://en.wikipedia.org/wiki/General_feature_format">@Wiki</a>
</li>
<li>
<strong>WIG</strong>: The WIG (wiggle) format is designed for display of dense continuous data such as probability scores. Wiggle data elements must be equally sized; if you need to display continuous data that is sparse or contains elements of varying size, use the BedGraph format instead. <a href="https://genome.ucsc.edu/goldenpath/help/wiggle.html">@UCSC</a>, <a href="http://useast.ensembl.org/info/website/upload/wig.html">@Ensembl</a>
</li>
<li>
<strong><a href="http://samtools.github.io/hts-specs/SAMv1.pdf">SAM</a></strong>: The SAM Format is a text format for storing sequence data in a series of tab delimited ASCII columns. <a href="http://genome.sph.umich.edu/wiki/SAM">@Wiki</a>
</li>
<li>
<strong>VCF</strong>: The Variant Call Format (VCF) specifies the format of a text file used in bioinformatics for storing gene sequence variations. <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-40">v4.0@1000genomes</a>, <a href="https://en.wikipedia.org/wiki/Variant_Call_Format">@Wiki</a>
</li>
<li>
<strong><a href="https://wiki.nci.nih.gov/display/TCGA/Mutation+Annotation+Format+%28MAF%29+Specification+-+v2.4">MAF - Mutation Annotation Format</a></strong>: A Mutation Annotation Format (MAF) file (.maf) is a tab-delimited text file that lists mutations. <a href="https://www.biostars.org/p/69222/">Tutorial@Biostars</a>
</li>
<li>
<strong>pileup</strong>: Pileup format is a text-based format for summarizing the base calls of aligned reads to a reference sequence. <a href="https://en.wikipedia.org/wiki/Pileup_format">@Wiki</a>
</li>
<li>
<strong>fasta</strong>: FASTA format is a text-based format for representing either nucleotide sequences or peptide sequences, in which nucleotides or amino acids are represented using single-letter codes. <a href="https://en.wikipedia.org/wiki/FASTA_format">@Wiki</a>
</li>
</ul>

<h3>
<a id="tools-1" class="anchor" href="#tools-1" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tools</h3>

<ul>
<li>
<strong><a href="https://bedops.readthedocs.io/en/latest/">BEDOPS</a></strong>: the fast, highly scalable and easily-parallelizable genome analysis toolkit.</li>
<li>
<strong><a href="https://github.com/arq5x/bedtools2">bedtools</a></strong>: Collectively, the bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks.</li>
<li>
<strong><a href="https://github.com/CRG-Barcelona/bwtool">bwtool</a></strong>: bwtool is a command-line utility for bigWig files.</li>
<li>
<strong><a href="https://github.com/lomereiter/sambamba">sambamba</a></strong>: samtools functions with multi-threading support.</li>
<li>
<strong><a href="https://github.com/samtools/samtools">samtools</a></strong>: SAM Tools provide various utilities for manipulating alignments in the SAM/BAM format, including sorting, merging, indexing and generating alignments in a per-position format.</li>
<li>
<strong><a href="https://vcftools.github.io/index.html">VCFtools</a></strong>: A set of tools written in Perl and C++ for working with VCF files.</li>
<li>
<strong><a href="https://broadinstitute.github.io/picard/">PiCard</a></strong>: Picard is a set of command line tools for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.</li>
<li>
<strong><a href="https://github.com/ctsa/svtools">SVTools</a></strong>: Tools for processing and analyzing structural variants.</li>
</ul>

<h2>
<a id="mathstatistics" class="anchor" href="#mathstatistics" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Math/Statistics</h2>

<h3>
<a id="distributions" class="anchor" href="#distributions" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Distributions</h3>

<ul>
<li>
<strong>T Distribution</strong>: Student's t-distribution (or t-distribution) is any member of a family of continuous probability distributions that arises when estimating the mean of a normally distributed population in situations where the sample size is small and population standard deviation is unknown.</li>
<li>
<strong>Chi-Squred Distribution</strong>: The chi-squared distribution (also chi-square or χ²-distribution) with k degrees of freedom is the distribution of a sum of the squares of k independent standard normal random variables. <a href="https://en.wikipedia.org/wiki/Chi-squared_distribution">@Wiki</a> </li>
</ul>

<h3>
<a id="tests" class="anchor" href="#tests" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tests</h3>

<ul>
<li>
<strong>T Test</strong>: A t-test is any statistical hypothesis test in which the test statistic follows a Student's t-distribution under the null hypothesis. It can be used to determine if two sets of data are significantly different from each other. <a href="https://en.wikipedia.org/wiki/Student%27s_t-test">@Wiki</a>
</li>
<li>
<strong>Chi-Squared Test</strong>: also referred to as a {\displaystyle \chi ^{2}} \chi ^{2} test, is any statistical hypothesis test wherein the sampling distribution of the test statistic is a chi-square distribution when the null hypothesis is true. <a href="https://en.wikipedia.org/wiki/Chi-squared_test">@Wiki</a>
</li>
<li>
<strong><a href="http://software.broadinstitute.org/gsea/index.jsp">GSEA - Gene Set Enrichment Analysis</a></strong>: Gene Set Enrichment Analysis (GSEA) is a computational method that determines whether an a priori defined set of genes shows statistically significant, concordant differences between two biological states (e.g. phenotypes). </li>
<li>
<strong><a href="https://github.com/ctlab/fgsea">fgsea</a></strong>: An R-package for fast preranked gene set enrichment analysis (GSEA).</li>
</ul>

<h3>
<a id="machine-learning" class="anchor" href="#machine-learning" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Machine Learning</h3>

<ul>
<li>
<strong>SVM - Support Vector Machine</strong>: support vector machines (SVMs, also support vector networks) are supervised learning models with associated learning algorithms that analyze data used for classification and regression analysis. <a href="https://en.wikipedia.org/wiki/Support_vector_machine">@Wiki</a>
</li>
</ul>
        </section>

        <aside id="sidebar">
          <a href="https://github.com/pwwang/Bioinformatics-cheatsheet/zipball/master" class="button">
            <small>Download</small>
            .zip file
          </a>
          <a href="https://github.com/pwwang/Bioinformatics-cheatsheet/tarball/master" class="button">
            <small>Download</small>
            .tar.gz file
          </a>

          <p class="repo-owner"><a href="https://github.com/pwwang/Bioinformatics-cheatsheet"></a> is maintained by <a href="https://github.com/pwwang">pwwang</a>.</p>

          <p>This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the Architect theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.</p>
        </aside>
      </div>
    </div>

  
  </body>
</html>
