# Germline Mutations and Genetic Diseases
<!-- toc -->

## Related Terms
- **CNV - Copy-Number Variation**: Copy-number variations (CNVs) are a form of structural variation that manifest as deletions or duplications in the genome. [@Wiki](https://en.wikipedia.org/wiki/Copy-number_variation)(https://www.genome.gov/25520880/deoxyribonucleic-acid-dna-fact-sheet/)
- **SNP**: A single nucleotide polymorphism, often abbreviated to SNP (pronounced snip; plural snips), is a variation in a single nucleotide that occurs at a specific position in the genome, where each variation is present to some appreciable degree within a population (e.g. >1%). [@Wiki](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism), [@NIH](https://ghr.nlm.nih.gov/primer/genomicresearch/snp)
- **Genotype**: The genotype is the part (DNA sequence) of the genetic makeup of a cell, and therefore of an organism or individual, which determines a specific characteristic (phenotype) of that cell/organism/individual. [@Wiki](https://en.wikipedia.org/wiki/Genotype)
- **PheWAS**: Phenome-wide association studies (PheWAS) analyze many phenotypes compared to a single genetic variant (or other attribute). [@Tool](https://github.com/PheWAS/PheWAS)
- **Haplotype**: A haplotype (haploid genotype) is a group of genes in an organism that are inherited together from a single parent. A haplogroup is a group of similar haplotypes that share a common ancestor with a single-nucleotide polymorphism mutation. [@Wiki](https://en.wikipedia.org/wiki/Haplotype), [@Scitable](www.nature.com/scitable/definition/haplotype-haplotypes-142)
- **IBD - Identity By Descent**: A DNA segment is identical by state (IBS) in two or more individuals if they have identical nucleotide sequences in this segment. An IBS segment is identical by descent (IBD) in two or more individuals if they have inherited it from a common ancestor without recombination, that is, the segment has the same ancestral origin in these individuals. [@Wiki](https://en.wikipedia.org/wiki/Identity_by_descent)
- **Phasing/Haplotype Estimation**: In genetics, haplotype estimation (also known as "phasing") refers to the process of statistical estimation of haplotypes from genotype data.
- **ICD**: The International Statistical Classification of Diseases and Related Health Problems, usually called by the short-form name International Classification of Diseases (ICD), is the international "standard diagnostic tool for epidemiology, health management and clinical purposes". [@Wiki](https://en.wikipedia.org/wiki/International_Statistical_Classification_of_Diseases_and_Related_Health_Problems)
- **MAF - Minor Allele Frequency**: MAF refers to the frequency at which the second most common allele occurs in a given population. SNPs with a minor allele frequency of 5% or greater were targeted by the HapMap project.
- **LOH - Loss Of Heterozygosity**: Loss of heterozygosity (LOH) is a gross chromosomal event that results in loss of the entire gene and the surrounding chromosomal region. [@Wiki](https://en.wikipedia.org/wiki/Loss_of_heterozygosity)
- **Missense Mutation**: A missense mutation is a point mutation in which a single nucleotide change results in a codon that codes for a different amino acid. [@Wiki](https://en.wikipedia.org/wiki/Missense_mutation)
- **Nonsense Mutation**: A nonsense mutation is a point mutation in a sequence of DNA that results in a premature stop codon, or a nonsense codon in the transcribed mRNA, and in a truncated, incomplete, and usually nonfunctional protein product.
- **Rare Variant**: A rare functional variant is a genetic variant which alters gene function, and which occurs at low frequency in a population. [@Wiki](https://en.wikipedia.org/wiki/Rare_functional_variant)
- **Allele Frequency**: Allele frequency, or gene frequency, is the relative frequency of an allele (variant of a gene) at a particular locus in a population, expressed as a fraction or percentage. [@Wiki](https://en.wikipedia.org/wiki/Allele_frequency)
- **SV - Structural Variation**: Structural variation (also genomic structural variation) is the variation in structure of an organism's chromosome. It consists of many kinds of variation in the genome of one species, and usually includes microscopic and submicroscopic types, such as deletions, duplications, copy-number variants, insertions, inversions and translocations.
- **[Transition vs Transversion](https://www.mun.ca/biology/scarr/Transitions_vs_Transversions.html)**: DNA substitution mutations are of two types. Transitions are interchanges of two-ring purines (A <-> G) or of one-ring pyrimidines (C <-> T): they therefore involve bases of similar shape. Transversions are interchanges of purine for pyrimidine bases, which therefore involve exchange of one-ring and two-ring structures (A <-> C, A <-> T, G <-> C, G <-> T).
- **[IUPAC codes]**(http://www.bioinformatics.org/sms/iupac.html): The International Union of Pure and Applied Chemistry (IUPAC) has defined a standard representation of DNA bases by single characters that specify either a single base (e.g. G for guanine, A for adenine) or a set of bases (e.g. R for either G or A). UCSC uses these single character codes to represent multiple observed alleles of single-base polymorphisms. [@UCSC](http://genome.ucsc.edu/goldenPath/help/iupac.html)

## Genetic Variant/Disease Databases
- **[dbSNP](http://www.ncbi.nlm.nih.gov/SNP/)**: The Single Nucleotide Polymorphism Database (dbSNP) is a free public archive for genetic variation within and across different species developed and hosted by the National Center for Biotechnology Information (NCBI) in collaboration with the National Human Genome Research Institute (NHGRI).
- **[GWASdb](http://jjwanglab.org/gwasdb)**: GWASdb is an online bioinformatics database combines collections of GVs from GWAS and their comprehensive functional annotations, as well as disease classifications.
- **[GWAS Central](http://www.gwascentral.org/)**: GWAS Central provides a centralized compilation of summary level findings from genetic association studies, both large and small. 
- **[OMIM - Online Mendelian Inheritance in ManÂ®](http://www.omim.org/)**: An Online Catalog of Human Genes and Genetic Disorders 
- **[eMERGE](https://emerge.mc.vanderbilt.edu/)**: eMERGE is a national network that combines DNA biorepositories with electronic medical record (EMR) systems for large scale, high-throughput genetic research in support of implementing genomic medicine.
- **[International HapMap Project](https://www.genome.gov/10001688/international-hapmap-project/)**: The International HapMap Project was an organization that aimed to develop a haplotype map (HapMap) of the human genome, to describe the common patterns of human genetic variation. HapMap is used to find genetic variants affecting health, disease and responses to drugs and environmental factors.
- **[MutDB](http://www.mutdb.org/)**: A database for assessing the impact of genetic variants. [@Ref](http://nar.oxfordjournals.org/content/36/suppl_1/D815)
- **[The Genome of the Netherlands](http://www.nlgenome.nl/)**: 250 trios (father, mother and child) of Dutch descent.
- **[ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/)**: ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence. ClinVar thus facilitates access to and communication about the relationships asserted between human variation and observed health status, and the history of that interpretation. [@Ref](https://www.ncbi.nlm.nih.gov/pubmed/26582918)
- **[DGAP](http://www.bwhpathology.org/dgap/default.aspx)**: To identify apparently balanced chromosomal rearrangements in patients with multiple congenital anomalies and then to use these chromosomal rearrangements to map and identify genes that are disrupted or dysregulated in critical stages of human development.
- **[DECIPHER](https://decipher.sanger.ac.uk/index)**: DECIPHER (DatabasE of genomiC varIation and Phenotype in Humans using Ensembl Resources) is an interactive web-based database which incorporates a suite of tools designed to aid the interpretation of genomic variants.
- **[ClinGen](https://clinicalgenome.org/about/)**: ClinGen is a National Institutes of Health (NIH)-funded resource dedicated to building an authoritative central resource that defines the clinical relevance of genes and variants for use in precision medicine and research. 

## Tools
- **[SNPRelate](http://bioconductor.org/packages/release/bioc/html/SNPRelate.html)**: SNPRelate (high-performance computing R packages for multi-core symmetric multiprocessing computer architectures) to accelerate two key computations in GWAS: principal component analysis (PCA) and relatedness analysis using identity-by-descent (IBD) measures.
- **[EIGENSOFT](https://github.com/DReichLab/EIG)**: The EIGENSOFT package combines functionality from our population genetics methods [@Ref](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0020190), EIGENSTRAT stratification correction method [@Ref](http://www.nature.com/ng/journal/v38/n8/abs/ng1847.html), and FastPCA and PC-based selection statistic [@Ref](http://www.sciencedirect.com/science/article/pii/S0002929716000033).
- **[PLATO](http://www.ritchielab.psu.edu/software/plato-download)**: The PLatform for the Analysis, Translation, and Organization of large-scale data (PLATO) is a standalone program written in C++ that is designed to be a flexible and extensible analysis tool for a wide variety of genetic data.

## Mutation-Protein Structure Studies
- **[HotSpot3D](https://github.com/ding-lab/hotspot3d)**: This 3D proximity tool can be used to identify the mutation hotspots in the linear 1D sequence and correlates these hotspots with known or potential interacting domains based on both known intermolecular interactions and calculated proximity for potential intramolecular interactions. [@Ref](http://www.nature.com/ng/journal/v48/n8/full/ng.3586.html)
- **[MuPIT_Interactive](http://mupit.icm.jhu.edu/MuPIT_Interactive/)**: webserver for mapping variant positions to annotated, interactive 3D structures. [@Ref](http://link.springer.com/article/10.1007%2Fs00439-013-1325-0)
- **[Interactome3D](http://interactome3d.irbbarcelona.org/)**: Interactome3D is a web service for the structural annotation of protein-protein interaction networks. [@Ref](http://www.nature.com/nmeth/journal/v10/n1/full/nmeth.2300.html)
- **[CLUMP](https://github.com/KarchinLab/CLUMP)**: CLUMP (CLustering by Mutation Postion) is an unsupervised clustering of amino acid residue positions where variants occur, without any prior knowledge of their functional importance. [@Ref](http://www.pnas.org/content/112/40/E5486)