# Pathways
<!-- toc -->

## Related Terms
- **Pathway (Biological pathway)**: A biological pathway is a series of actions among molecules in a cell that leads to a certain product or a change in a cell. Such a pathway can trigger the assembly of new molecules, such as a fat or protein. Pathways can also turn genes on and off, or spur a cell to move. [@Wiki](https://en.wikipedia.org/wiki/Biological_pathway), [@NIH](https://www.genome.gov/27530687/biological-pathways-fact-sheet/)

## Pathway Databases
- **[ConsensusPathDB](http://consensuspathdb.org/)**: ConsensusPathDB is a database that integrates different types of functional interactions between physical entities in the cell like genes, RNA, proteins, protein complexes and metabolites in order to assemble a more complete and a less biased picture of cellular biology.
- **[KEGG PATHWAY](www.genome.jp/kegg/pathway.html)**: KEGG PATHWAY is a collection of manually drawn pathway maps representing our knowledge on the molecular interaction and reaction networks for different aspects.
- **[MetaCyc](http://metacyc.org/)**: MetaCyc is a curated database of experimentally elucidated metabolic pathways from all domains of life.
- **[MouseCyc](http://www.informatics.jax.org/pathways.shtml)**: MouseCyc is a database of curated biochemical pathways data for the laboratory mouse that can be integrated with functional and phenotypic data from MGI.
- **[PANTHER Pathway](http://www.pantherdb.org/pathway/)**: PANTHER Pathway consists of over 177, primarily signaling, pathways, each with subfamilies and protein sequences mapped to individual pathway components. 
- **[Pathway Commons](http://pathwaycommons.org/)**: Pathway Commons aims to store and disseminate knowledge about biological pathways. Information is sourced from public pathway databases and is readily searched, visualized, and downloaded.
- **[Reactome](http://reactome.org/)**: Reactome is a free, open-source, curated and peer reviewed pathway database.
- **[PLANTCYC](http://www.plantcyc.org/)**: PlantCyc is a metabolic pathway reference database containing more than 800 pathways and their catalytic enzymes and genes, as well as compounds from over 350 plant species. It includes: [AraCyc](http://www.plantcyc.org/databases/aracyc/14.0)(Arabidopsis thaliana col), [BarleyCyc](http://www.plantcyc.org/databases/barleycyc/4.0)(Hordeum vulgare), [BrachypodiumCyc](http://www.plantcyc.org/databases/brachypodiumcyc/4.0)(Brachypodium distachyon), [CassavaCyc](http://www.plantcyc.org/databases/cassavacyc/6.0)(Manihot esculenta esculenta), [ChineseCabbageCyc](http://www.plantcyc.org/databases/chinesecabbagecyc/4.0)(Brassica rapa ssp. pekinensis), [ChlamyCyc](http://www.plantcyc.org/databases/chlamycyc/6.0)(Chlamydomonas reinhardtii), [CornCyc](http://www.plantcyc.org/databases/corncyc/7.0)(Zea	mays mays), [GrapeCyc](http://www.plantcyc.org/databases/grapecyc/6.0)(Vitis vinifera), [MossCyc](http://www.plantcyc.org/databases/mosscyc/5.0)(Physcomitrella patens), [OryzaCyc](http://www.plantcyc.org/databases/oryzacyc/4.0)(Oryza sativa japonica group), [PapayaCyc](http://www.plantcyc.org/databases/papayacyc/5.0)(Carica papaya), [PoplarCyc](http://www.plantcyc.org/databases/poplarcyc/9.0)(Populus trichocarpa, other Populus species and hybrids), [PotatoCyc](http://www.plantcyc.org/databases/potatocyc/3.0)(Solanum tuberosum), [SelaginellaCyc](http://www.plantcyc.org/databases/selaginellacyc/5.0)(Selaginella moellendorffii), [SetariaCyc](http://www.plantcyc.org/databases/setariacyc/4.0)(Setaria italica), [SorghumBicolorCyc](http://www.plantcyc.org/databases/sorghumbicolorcyc/4.0)(Sorghum bicolor), [SoyCyc](http://www.plantcyc.org/databases/soycyc/7.0)(Glycine max), [SpirodelaCyc](http://www.plantcyc.org/databases/spirodelacyc/2.0)(Spirodela polyrhiza), [SwitchgrassCyc](http://www.plantcyc.org/databases/switchgrasscyc/4.0)(Panicum virgatum), [TomatoCyc](http://www.plantcyc.org/databases/tomatocyc/2.0)(Solanum lycopersicum), [WheatACyc](http://www.plantcyc.org/databases/wheatacyc/2.0)(Triticum urartu), [WheatDCyc](http://www.plantcyc.org/databases/wheatdcyc/2.0)(Aegilops tauschii)
- **[SignaLink](http://signalink.org/)**: An integrated resource to analyze signaling pathway cross-talks, transcription factors, miRNAs and regulatory enzymes.
- **[SMPDB](http://smpdb.ca/)**: SMPDB (The Small Molecule Pathway Database) is an interactive, visual database containing more than 618 small molecule pathways found in humans. More than 70% of these pathways (>433) are not found in any other pathway database.
- **[Yeast Pathways Database](http://pathway.yeastgenome.org/)**: The Yeast Pathways Database is a collection of manually curated metabolic pathways and enzymes of Saccharomyces cerevisiae.

## Pathway Predictions
- **[PIUMet](http://fraenkel-nsf.csbi.mit.edu/PIUMet/)**: Inferring Disease-Modifying Pathways and Hidden Components via Integrative Analysis of Metabolite Features with Various Omic Data. [@Ref](http://www.nature.com/nmeth/journal/v13/n9/pdf/nmeth.3940.pdf)

## Pathway/Network analysis/visualizers
- **[HotNet2](https://github.com/raphael-group/hotnet2)**: HotNet2 is an algorithm for finding significantly altered subnetworks in a large gene interaction network. While originally developed for use with cancer mutation data, the current release also supports any application in which meaningful scores can be assigned to genes in the network. [@Ref](http://www.nature.com/ng/journal/v47/n2/full/ng.3168.html)
- **[CellMaps](https://github.com/opencb/cell-maps/wiki)**: CellMaps is an open source HTML5 web-based application that allows researchers to easily model, visualize, integrate data and analyse biological networks inside a web browser. 