# Transcription Factor - Target
<!-- toc -->

## Related Terms
- **TF - Transcription Factor**: Transcription factors are proteins that control which genes are turned on or off in the genome. They do so by binding to DNA and other proteins. [@Wiki](https://en.wikipedia.org/wiki/Transcription_factor), [@BroadInstitute](https://www.broadinstitute.org/education/glossary/transcription-factor), [@Scitable](http://www.nature.com/scitable/definition/general-transcription-factor-transcription-factor-167)
- **PWM - Position Weight Matrix/PSWM/PSSM**: A position weight matrix (PWM), also known as a position-specific weight matrix (PSWM) or position-specific scoring matrix (PSSM), is a commonly used representation of motifs (patterns) in biological sequences. [@Wiki](https://en.wikipedia.org/wiki/Position_weight_matrix)
- **TFBS - Transcription Factor Binding Site/DNA Binding Site**: DNA binding sites are a type of binding site found in DNA where other molecules may bind. [@Wiki](https://en.wikipedia.org/wiki/DNA_binding_site)
- **DNA Sequence Motif**: Sequence motifs are short, recurring patterns in DNA that are presumed to have a biological function. [@Nature Biotechnology](http://www.nature.com/nbt/journal/v24/n4/full/nbt0406-423.html), [@Wiki](https://en.wikipedia.org/wiki/Sequence_motif)
- **Transcription**: Transcription is the first step of gene expression, in which a particular segment of DNA is copied into RNA (mRNA) by the enzyme RNA polymerase. [@Wiki](https://en.wikipedia.org/wiki/Transcription_(genetics))

## Transcription Factor Databases
- **[AnimalTFDB](http://bioinfo.life.hust.edu.cn/AnimalTFDB/)**: AnimalTFDB is a comprehensive database including classification and annotation of genome-wide transcription factors (TFs), transcription co-factors and chromatin remodeling factors in 65 animal genomes. 
- **[DBD](www.transcriptionfactor.org/)**: DBD is a database of predicted transcription factors in completely sequenced genomes.
- **[PlantTFDB](http://planttfdb.cbi.pku.edu.cn/)**: Plant transcription factor database, a portal for the functional and evolutionary study of plant transcription factors.
- **[TFCat](www.tfcat.ca/)**: TFCat: The curated catalog of mouse and human transcription factors.
- **[TFdb](http://genome.gsc.riken.jp/TFdb/)**: The Mouse transcription factor database (TFdb) is a database containing mouse transcription factor genes and their related genes.

## TFBS/TF Binding Motif/TF Target Databases
- **[Cistrome DB](http://dc2.cistrome.org/)**: Cistrome DB is a comprehensive resource of hg38 and mm10 ChIP-seq data collection. Here is a brief introduction about the workflow of [ChiLin](http://cistrome.org/chilin/).
- **[CollecTF](http://www.collectf.org/browse/home/)**: CollecTF is a database of transcription factor binding sites (TFBS) in the Bacteria domain.
- **[CTCFBSDB](http://insulatordb.uthsc.edu/)**: A database for CTCF binding sites and genome organization.
- **[FactorBook](http://www.factorbook.org/)**: This website organizes the analysis results of ENCODE TF ChIP-seq data, integrated with other ENCODE data such as ChIP-seq of histone marks and nucleosome occupancy.
- **[footprintDB](http://floresta.eead.csic.es/footprintdb/index.php)**: footprintDB is a web server for assigning putative cis DNA motifs to input transcription factors (TFs) and conversely for predicting which TFs that might recognize input DNA motifs.
- **[hmChIP](http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl)**: hmChIP is a database of genome-wide chromatin immu-noprecipitation (ChIP) data in human and mouse.
- **[HOCOMOCO](http://hocomoco.autosome.ru/)**: HOmo sapiens COmprehensive MOdel COllection (HOCOMOCO) contains transcription factor (TF) binding models represented as classic Position Weight Matrices (PWMs, also known as Position-Specific Scoring Matrices, PSSMs) and precalculated score thresholds.
- **[HOMER Motif Database](http://homer.salk.edu/homer/motif/motifDatabase.html)**: This database is maintained as part of HOMER and is mostly based on the analysis of public ChIP-Seq data sets.
- **[hPDI](http://bioinfo.wilmer.jhu.edu/PDI/)**: The hPDI database holds experimental protein-DNA interaction data for humans identified by protein microarray assays.
- **[HTRIdb](http://www.lbbc.ibb.unesp.br/htri)**: Human Transcriptional Regulation Interaction Database.
- **[JASPAR](jaspar.binf.ku.dk/)**: The high-quality transcription factor binding profile database.
- **[MAPPER](http://genome.ufl.edu/mapper/)**: MAPPER is a platform for the computational identification of transcription factor binding sites (TFBSs) in multiple genomes, that combines TRANSFAC® and JASPAR data with the search power of profile hidden Markov models (HMMs).
- **[MotifMap](http://motifmap.ics.uci.edu/)**: The MotifMap system provides comprehensive maps of candidate regulatory elements encoded in the genomes of model species using databases of transcription factor binding motifs, refined genome alignments, and a comparative genomic statistical approach - Bayesian Branch Length Score.
- **[oPOSSUM](http://opossum.cisreg.ca/oPOSSUM3/)**: oPOSSUM is a web-based system for the detection of over-represented conserved transcription factor binding sites and binding site combinations in sets of genes or sequences.
- **[SwissRegulon](http://swissregulon.unibas.ch/fcgi/sr/swissregulon)**: Swissregulon Database contains genome-wide annotations of regulatory sites.
- **[TFBSshape](http://rohslab.cmb.usc.edu/TFBSshape/)**: TFBSshape provides DNA shape features for transcription factor binding sites (TFBSs) that in addtion to sequence features, usually in the form of position weight matrices (PWMs), characterize DNA binding specificities of transcription factors (TFs) from 23 different species.
- **[TRANSFAC](http://www.biobase-international.com/product/transcription-factor-binding-sites)**: TRANSFAC® is a unique knowledge-base containing published data on eukaryotic transcription factors and miRNAs, their experimentally-proven binding sites, and regulated genes.
- **[UniPROBE](http://the_brain.bwh.harvard.edu/uniprobe/index.php)**: The UniPROBE (Universal PBM Resource for Oligonucleotide Binding Evaluation) database hosts data generated by universal protein binding microarray (PBM) technology on the in vitro DNA binding specificities of proteins. 

## TFBS Prediction
- **[DeepSEA](http://deepsea.princeton.edu/job/analysis/create/)**: DeepSEA is a deep learning-based algorithmic framework for predicting the chromatin effects of sequence alterations with single nucleotide sensitivity. DeepSEA can accurately predict the epigenetic state of a sequence, including transcription factors binding, DNase I sensitivities and histone marks in multiple cell types, and further utilize this capability to predict the chromatin effects of sequence variants and prioritize regulatory variants. [@Ref](http://www.nature.com/nmeth/journal/v12/n10/full/nmeth.3547.html)

## Protein-DNA Interaction Detection Methods
- **PBM - Protein Binding Microarray**: //TODO
- **ChIP**: Chromatin Immunoprecipitation (ChIP) is a type of immunoprecipitation experimental technique used to investigate the interaction between proteins and DNA in the cell. [@Wiki](https://en.wikipedia.org/wiki/Chromatin_immunoprecipitation)
- **ChIP-seq**: ChIP-sequencing, also known as ChIP-seq, is a method used to analyze protein interactions with DNA. ChIP-seq combines chromatin immunoprecipitation (ChIP) with massively parallel DNA sequencing to identify the binding sites of DNA-associated proteins. [@Wiki](https://en.wikipedia.org/wiki/ChIP-sequencing)
- **ChIP-chip**: ChIP-chip (also known as ChIP-on-chip) is a technology that combines chromatin immunoprecipitation ('ChIP') with DNA microarray ("chip"). Like regular ChIP, ChIP-on-chip is used to investigate interactions between proteins and DNA in vivo. [@Wiki](https://en.wikipedia.org/wiki/ChIP-on-chip)